[32mINFO[0m:     [28-12-2023 18:00:33] Set chroma_server_nofile to 65535
[32mINFO[0m:     [28-12-2023 18:03:32] Set chroma_server_nofile to 65535
[32mINFO[0m:     [28-12-2023 18:07:05] Set chroma_server_nofile to 65535
[32mINFO[0m:     [28-12-2023 18:19:24] Set chroma_server_nofile to 65535
[32mINFO[0m:     [28-12-2023 18:21:26] Set chroma_server_nofile to 65535
[32mINFO[0m:     [28-12-2023 18:26:09] Set chroma_server_nofile to 65535
[32mINFO[0m:     [28-12-2023 18:31:18] Set chroma_server_nofile to 65535
[32mINFO[0m:     [28-12-2023 18:33:10] Set chroma_server_nofile to 65535
[32mINFO[0m:     [28-12-2023 18:34:23] Set chroma_server_nofile to 65535
[32mINFO[0m:     [28-12-2023 18:34:41] Set chroma_server_nofile to 65535
[32mINFO[0m:     [28-12-2023 18:34:53] Set chroma_server_nofile to 65535
[32mINFO[0m:     [28-12-2023 18:34:54] Anonymized telemetry enabled. See                     https://docs.trychroma.com/telemetry for more information.
[36mDEBUG[0m:    [28-12-2023 18:34:54] Starting component System
[36mDEBUG[0m:    [28-12-2023 18:34:54] Starting component OpenTelemetryClient
[36mDEBUG[0m:    [28-12-2023 18:34:54] Starting component SimpleAssignmentPolicy
[36mDEBUG[0m:    [28-12-2023 18:34:54] Starting component SqliteDB
[36mDEBUG[0m:    [28-12-2023 18:34:54] Starting component Posthog
[36mDEBUG[0m:    [28-12-2023 18:34:54] Starting component LocalSegmentManager
[36mDEBUG[0m:    [28-12-2023 18:34:54] Starting component SegmentAPI
[32mINFO[0m:     [28-12-2023 18:34:54] Started server process [[36m53169[0m]
[32mINFO[0m:     [28-12-2023 18:34:54] Waiting for application startup.
[32mINFO[0m:     [28-12-2023 18:34:54] Application startup complete.
[32mINFO[0m:     [28-12-2023 18:34:54] Uvicorn running on [1mhttp://localhost:8000[0m (Press CTRL+C to quit)
[32mINFO[0m:     [28-12-2023 18:35:12] ::1:65083 - "POST /api/v1/collections?tenant=default_tenant&database=default_database HTTP/1.1" 200
[31mERROR[0m:    [28-12-2023 18:44:17] Collection movie_reviews already exists
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/anyio/streams/memory.py", line 94, in receive
    return self.receive_nowait()
  File "/usr/local/lib/python3.9/site-packages/anyio/streams/memory.py", line 89, in receive_nowait
    raise WouldBlock
anyio.WouldBlock

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/base.py", line 78, in call_next
    message = await recv_stream.receive()
  File "/usr/local/lib/python3.9/site-packages/anyio/streams/memory.py", line 114, in receive
    raise EndOfStream
anyio.EndOfStream

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/chromadb/server/fastapi/__init__.py", line 80, in catch_exceptions_middleware
    return await call_next(request)
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/base.py", line 84, in call_next
    raise app_exc
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/base.py", line 70, in coro
    await self.app(scope, receive_or_disconnect, send_no_error)
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/base.py", line 108, in __call__
    response = await self.dispatch_func(request, call_next)
  File "/usr/local/lib/python3.9/site-packages/chromadb/server/fastapi/__init__.py", line 94, in check_http_version_middleware
    return await call_next(request)
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/base.py", line 84, in call_next
    raise app_exc
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/base.py", line 70, in coro
    await self.app(scope, receive_or_disconnect, send_no_error)
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/exceptions.py", line 79, in __call__
    raise exc
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/exceptions.py", line 68, in __call__
    await self.app(scope, receive, sender)
  File "/usr/local/lib/python3.9/site-packages/fastapi/middleware/asyncexitstack.py", line 20, in __call__
    raise e
  File "/usr/local/lib/python3.9/site-packages/fastapi/middleware/asyncexitstack.py", line 17, in __call__
    await self.app(scope, receive, send)
  File "/usr/local/lib/python3.9/site-packages/starlette/routing.py", line 718, in __call__
    await route.handle(scope, receive, send)
  File "/usr/local/lib/python3.9/site-packages/starlette/routing.py", line 276, in handle
    await self.app(scope, receive, send)
  File "/usr/local/lib/python3.9/site-packages/starlette/routing.py", line 66, in app
    response = await func(request)
  File "/usr/local/lib/python3.9/site-packages/fastapi/routing.py", line 273, in app
    raw_response = await run_endpoint_function(
  File "/usr/local/lib/python3.9/site-packages/fastapi/routing.py", line 192, in run_endpoint_function
    return await run_in_threadpool(dependant.call, **values)
  File "/usr/local/lib/python3.9/site-packages/starlette/concurrency.py", line 41, in run_in_threadpool
    return await anyio.to_thread.run_sync(func, *args)
  File "/usr/local/lib/python3.9/site-packages/anyio/to_thread.py", line 31, in run_sync
    return await get_asynclib().run_sync_in_worker_thread(
  File "/usr/local/lib/python3.9/site-packages/anyio/_backends/_asyncio.py", line 937, in run_sync_in_worker_thread
    return await future
  File "/usr/local/lib/python3.9/site-packages/anyio/_backends/_asyncio.py", line 867, in run
    result = context.run(func, *args)
  File "/usr/local/lib/python3.9/site-packages/chromadb/telemetry/opentelemetry/__init__.py", line 127, in wrapper
    return f(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/chromadb/auth/fastapi.py", line 234, in wrapped
    return f(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/chromadb/server/fastapi/__init__.py", line 408, in create_collection
    return self._api.create_collection(
  File "/usr/local/lib/python3.9/site-packages/chromadb/telemetry/opentelemetry/__init__.py", line 127, in wrapper
    return f(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/chromadb/api/segment.py", line 166, in create_collection
    coll, created = self._sysdb.create_collection(
  File "/usr/local/lib/python3.9/site-packages/chromadb/telemetry/opentelemetry/__init__.py", line 127, in wrapper
    return f(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/chromadb/db/mixins/sysdb.py", line 225, in create_collection
    raise UniqueConstraintError(f"Collection {name} already exists")
chromadb.db.base.UniqueConstraintError: Collection movie_reviews already exists
[32mINFO[0m:     [28-12-2023 18:44:17] ::1:65121 - "POST /api/v1/collections?tenant=default_tenant&database=default_database HTTP/1.1" 500
[31mERROR[0m:    [28-12-2023 18:48:09] Collection movie_reviews already exists
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/anyio/streams/memory.py", line 94, in receive
    return self.receive_nowait()
  File "/usr/local/lib/python3.9/site-packages/anyio/streams/memory.py", line 89, in receive_nowait
    raise WouldBlock
anyio.WouldBlock

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/base.py", line 78, in call_next
    message = await recv_stream.receive()
  File "/usr/local/lib/python3.9/site-packages/anyio/streams/memory.py", line 114, in receive
    raise EndOfStream
anyio.EndOfStream

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/chromadb/server/fastapi/__init__.py", line 80, in catch_exceptions_middleware
    return await call_next(request)
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/base.py", line 84, in call_next
    raise app_exc
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/base.py", line 70, in coro
    await self.app(scope, receive_or_disconnect, send_no_error)
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/base.py", line 108, in __call__
    response = await self.dispatch_func(request, call_next)
  File "/usr/local/lib/python3.9/site-packages/chromadb/server/fastapi/__init__.py", line 94, in check_http_version_middleware
    return await call_next(request)
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/base.py", line 84, in call_next
    raise app_exc
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/base.py", line 70, in coro
    await self.app(scope, receive_or_disconnect, send_no_error)
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/exceptions.py", line 79, in __call__
    raise exc
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/exceptions.py", line 68, in __call__
    await self.app(scope, receive, sender)
  File "/usr/local/lib/python3.9/site-packages/fastapi/middleware/asyncexitstack.py", line 20, in __call__
    raise e
  File "/usr/local/lib/python3.9/site-packages/fastapi/middleware/asyncexitstack.py", line 17, in __call__
    await self.app(scope, receive, send)
  File "/usr/local/lib/python3.9/site-packages/starlette/routing.py", line 718, in __call__
    await route.handle(scope, receive, send)
  File "/usr/local/lib/python3.9/site-packages/starlette/routing.py", line 276, in handle
    await self.app(scope, receive, send)
  File "/usr/local/lib/python3.9/site-packages/starlette/routing.py", line 66, in app
    response = await func(request)
  File "/usr/local/lib/python3.9/site-packages/fastapi/routing.py", line 273, in app
    raw_response = await run_endpoint_function(
  File "/usr/local/lib/python3.9/site-packages/fastapi/routing.py", line 192, in run_endpoint_function
    return await run_in_threadpool(dependant.call, **values)
  File "/usr/local/lib/python3.9/site-packages/starlette/concurrency.py", line 41, in run_in_threadpool
    return await anyio.to_thread.run_sync(func, *args)
  File "/usr/local/lib/python3.9/site-packages/anyio/to_thread.py", line 31, in run_sync
    return await get_asynclib().run_sync_in_worker_thread(
  File "/usr/local/lib/python3.9/site-packages/anyio/_backends/_asyncio.py", line 937, in run_sync_in_worker_thread
    return await future
  File "/usr/local/lib/python3.9/site-packages/anyio/_backends/_asyncio.py", line 867, in run
    result = context.run(func, *args)
  File "/usr/local/lib/python3.9/site-packages/chromadb/telemetry/opentelemetry/__init__.py", line 127, in wrapper
    return f(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/chromadb/auth/fastapi.py", line 234, in wrapped
    return f(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/chromadb/server/fastapi/__init__.py", line 408, in create_collection
    return self._api.create_collection(
  File "/usr/local/lib/python3.9/site-packages/chromadb/telemetry/opentelemetry/__init__.py", line 127, in wrapper
    return f(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/chromadb/api/segment.py", line 166, in create_collection
    coll, created = self._sysdb.create_collection(
  File "/usr/local/lib/python3.9/site-packages/chromadb/telemetry/opentelemetry/__init__.py", line 127, in wrapper
    return f(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/chromadb/db/mixins/sysdb.py", line 225, in create_collection
    raise UniqueConstraintError(f"Collection {name} already exists")
chromadb.db.base.UniqueConstraintError: Collection movie_reviews already exists
[32mINFO[0m:     [28-12-2023 18:48:09] ::1:65179 - "POST /api/v1/collections?tenant=default_tenant&database=default_database HTTP/1.1" 500
[32mINFO[0m:     [28-12-2023 18:48:19] ::1:65180 - "GET /api/v1/collections?tenant=default_tenant&database=default_database HTTP/1.1" 200
[31mERROR[0m:    [28-12-2023 18:50:08] Collection movie_reviews already exists
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/anyio/streams/memory.py", line 94, in receive
    return self.receive_nowait()
  File "/usr/local/lib/python3.9/site-packages/anyio/streams/memory.py", line 89, in receive_nowait
    raise WouldBlock
anyio.WouldBlock

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/base.py", line 78, in call_next
    message = await recv_stream.receive()
  File "/usr/local/lib/python3.9/site-packages/anyio/streams/memory.py", line 114, in receive
    raise EndOfStream
anyio.EndOfStream

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/chromadb/server/fastapi/__init__.py", line 80, in catch_exceptions_middleware
    return await call_next(request)
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/base.py", line 84, in call_next
    raise app_exc
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/base.py", line 70, in coro
    await self.app(scope, receive_or_disconnect, send_no_error)
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/base.py", line 108, in __call__
    response = await self.dispatch_func(request, call_next)
  File "/usr/local/lib/python3.9/site-packages/chromadb/server/fastapi/__init__.py", line 94, in check_http_version_middleware
    return await call_next(request)
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/base.py", line 84, in call_next
    raise app_exc
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/base.py", line 70, in coro
    await self.app(scope, receive_or_disconnect, send_no_error)
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/exceptions.py", line 79, in __call__
    raise exc
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/exceptions.py", line 68, in __call__
    await self.app(scope, receive, sender)
  File "/usr/local/lib/python3.9/site-packages/fastapi/middleware/asyncexitstack.py", line 20, in __call__
    raise e
  File "/usr/local/lib/python3.9/site-packages/fastapi/middleware/asyncexitstack.py", line 17, in __call__
    await self.app(scope, receive, send)
  File "/usr/local/lib/python3.9/site-packages/starlette/routing.py", line 718, in __call__
    await route.handle(scope, receive, send)
  File "/usr/local/lib/python3.9/site-packages/starlette/routing.py", line 276, in handle
    await self.app(scope, receive, send)
  File "/usr/local/lib/python3.9/site-packages/starlette/routing.py", line 66, in app
    response = await func(request)
  File "/usr/local/lib/python3.9/site-packages/fastapi/routing.py", line 273, in app
    raw_response = await run_endpoint_function(
  File "/usr/local/lib/python3.9/site-packages/fastapi/routing.py", line 192, in run_endpoint_function
    return await run_in_threadpool(dependant.call, **values)
  File "/usr/local/lib/python3.9/site-packages/starlette/concurrency.py", line 41, in run_in_threadpool
    return await anyio.to_thread.run_sync(func, *args)
  File "/usr/local/lib/python3.9/site-packages/anyio/to_thread.py", line 31, in run_sync
    return await get_asynclib().run_sync_in_worker_thread(
  File "/usr/local/lib/python3.9/site-packages/anyio/_backends/_asyncio.py", line 937, in run_sync_in_worker_thread
    return await future
  File "/usr/local/lib/python3.9/site-packages/anyio/_backends/_asyncio.py", line 867, in run
    result = context.run(func, *args)
  File "/usr/local/lib/python3.9/site-packages/chromadb/telemetry/opentelemetry/__init__.py", line 127, in wrapper
    return f(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/chromadb/auth/fastapi.py", line 234, in wrapped
    return f(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/chromadb/server/fastapi/__init__.py", line 408, in create_collection
    return self._api.create_collection(
  File "/usr/local/lib/python3.9/site-packages/chromadb/telemetry/opentelemetry/__init__.py", line 127, in wrapper
    return f(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/chromadb/api/segment.py", line 166, in create_collection
    coll, created = self._sysdb.create_collection(
  File "/usr/local/lib/python3.9/site-packages/chromadb/telemetry/opentelemetry/__init__.py", line 127, in wrapper
    return f(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/chromadb/db/mixins/sysdb.py", line 225, in create_collection
    raise UniqueConstraintError(f"Collection {name} already exists")
chromadb.db.base.UniqueConstraintError: Collection movie_reviews already exists
[32mINFO[0m:     [28-12-2023 18:50:08] ::1:65198 - "POST /api/v1/collections?tenant=default_tenant&database=default_database HTTP/1.1" 500
[32mINFO[0m:     [28-12-2023 18:55:52] Shutting down
[32mINFO[0m:     [28-12-2023 18:55:52] Waiting for application shutdown.
[32mINFO[0m:     [28-12-2023 18:55:52] Application shutdown complete.
[32mINFO[0m:     [28-12-2023 18:55:52] Finished server process [[36m53169[0m]
[32mINFO[0m:     [28-12-2023 18:55:55] Set chroma_server_nofile to 65535
[32mINFO[0m:     [28-12-2023 18:55:55] Anonymized telemetry enabled. See                     https://docs.trychroma.com/telemetry for more information.
[36mDEBUG[0m:    [28-12-2023 18:55:55] Starting component System
[36mDEBUG[0m:    [28-12-2023 18:55:55] Starting component OpenTelemetryClient
[36mDEBUG[0m:    [28-12-2023 18:55:55] Starting component SimpleAssignmentPolicy
[36mDEBUG[0m:    [28-12-2023 18:55:55] Starting component SqliteDB
[36mDEBUG[0m:    [28-12-2023 18:55:55] Starting component Posthog
[36mDEBUG[0m:    [28-12-2023 18:55:55] Starting component LocalSegmentManager
[36mDEBUG[0m:    [28-12-2023 18:55:55] Starting component SegmentAPI
[32mINFO[0m:     [28-12-2023 18:55:55] Started server process [[36m56781[0m]
[32mINFO[0m:     [28-12-2023 18:55:55] Waiting for application startup.
[32mINFO[0m:     [28-12-2023 18:55:55] Application startup complete.
[32mINFO[0m:     [28-12-2023 18:55:55] Uvicorn running on [1mhttp://localhost:8000[0m (Press CTRL+C to quit)
[32mINFO[0m:     [28-12-2023 18:56:18] ::1:65236 - "POST /api/v1/collections?tenant=default_tenant&database=default_database HTTP/1.1" 200
[32mINFO[0m:     [28-12-2023 18:56:18] ::1:65237 - "GET /api/v1/collections/movie_reviews?tenant=default_tenant&database=default_database HTTP/1.1" 200
[31mERROR[0m:    [29-12-2023 10:21:24] Collection movie_reviews already exists
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/anyio/streams/memory.py", line 94, in receive
    return self.receive_nowait()
  File "/usr/local/lib/python3.9/site-packages/anyio/streams/memory.py", line 89, in receive_nowait
    raise WouldBlock
anyio.WouldBlock

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/base.py", line 78, in call_next
    message = await recv_stream.receive()
  File "/usr/local/lib/python3.9/site-packages/anyio/streams/memory.py", line 114, in receive
    raise EndOfStream
anyio.EndOfStream

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/chromadb/server/fastapi/__init__.py", line 80, in catch_exceptions_middleware
    return await call_next(request)
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/base.py", line 84, in call_next
    raise app_exc
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/base.py", line 70, in coro
    await self.app(scope, receive_or_disconnect, send_no_error)
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/base.py", line 108, in __call__
    response = await self.dispatch_func(request, call_next)
  File "/usr/local/lib/python3.9/site-packages/chromadb/server/fastapi/__init__.py", line 94, in check_http_version_middleware
    return await call_next(request)
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/base.py", line 84, in call_next
    raise app_exc
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/base.py", line 70, in coro
    await self.app(scope, receive_or_disconnect, send_no_error)
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/exceptions.py", line 79, in __call__
    raise exc
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/exceptions.py", line 68, in __call__
    await self.app(scope, receive, sender)
  File "/usr/local/lib/python3.9/site-packages/fastapi/middleware/asyncexitstack.py", line 20, in __call__
    raise e
  File "/usr/local/lib/python3.9/site-packages/fastapi/middleware/asyncexitstack.py", line 17, in __call__
    await self.app(scope, receive, send)
  File "/usr/local/lib/python3.9/site-packages/starlette/routing.py", line 718, in __call__
    await route.handle(scope, receive, send)
  File "/usr/local/lib/python3.9/site-packages/starlette/routing.py", line 276, in handle
    await self.app(scope, receive, send)
  File "/usr/local/lib/python3.9/site-packages/starlette/routing.py", line 66, in app
    response = await func(request)
  File "/usr/local/lib/python3.9/site-packages/fastapi/routing.py", line 273, in app
    raw_response = await run_endpoint_function(
  File "/usr/local/lib/python3.9/site-packages/fastapi/routing.py", line 192, in run_endpoint_function
    return await run_in_threadpool(dependant.call, **values)
  File "/usr/local/lib/python3.9/site-packages/starlette/concurrency.py", line 41, in run_in_threadpool
    return await anyio.to_thread.run_sync(func, *args)
  File "/usr/local/lib/python3.9/site-packages/anyio/to_thread.py", line 31, in run_sync
    return await get_asynclib().run_sync_in_worker_thread(
  File "/usr/local/lib/python3.9/site-packages/anyio/_backends/_asyncio.py", line 937, in run_sync_in_worker_thread
    return await future
  File "/usr/local/lib/python3.9/site-packages/anyio/_backends/_asyncio.py", line 867, in run
    result = context.run(func, *args)
  File "/usr/local/lib/python3.9/site-packages/chromadb/telemetry/opentelemetry/__init__.py", line 127, in wrapper
    return f(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/chromadb/auth/fastapi.py", line 234, in wrapped
    return f(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/chromadb/server/fastapi/__init__.py", line 408, in create_collection
    return self._api.create_collection(
  File "/usr/local/lib/python3.9/site-packages/chromadb/telemetry/opentelemetry/__init__.py", line 127, in wrapper
    return f(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/chromadb/api/segment.py", line 166, in create_collection
    coll, created = self._sysdb.create_collection(
  File "/usr/local/lib/python3.9/site-packages/chromadb/telemetry/opentelemetry/__init__.py", line 127, in wrapper
    return f(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/chromadb/db/mixins/sysdb.py", line 225, in create_collection
    raise UniqueConstraintError(f"Collection {name} already exists")
chromadb.db.base.UniqueConstraintError: Collection movie_reviews already exists
[32mINFO[0m:     [29-12-2023 10:21:24] ::1:52021 - "POST /api/v1/collections?tenant=default_tenant&database=default_database HTTP/1.1" 500
[32mINFO[0m:     [29-12-2023 10:21:39] Shutting down
[32mINFO[0m:     [29-12-2023 10:21:39] Waiting for application shutdown.
[32mINFO[0m:     [29-12-2023 10:21:39] Application shutdown complete.
[32mINFO[0m:     [29-12-2023 10:21:39] Finished server process [[36m56781[0m]
[32mINFO[0m:     [29-12-2023 10:21:42] Set chroma_server_nofile to 65535
[32mINFO[0m:     [29-12-2023 10:21:42] Anonymized telemetry enabled. See                     https://docs.trychroma.com/telemetry for more information.
[36mDEBUG[0m:    [29-12-2023 10:21:42] Starting component System
[36mDEBUG[0m:    [29-12-2023 10:21:42] Starting component OpenTelemetryClient
[36mDEBUG[0m:    [29-12-2023 10:21:42] Starting component SimpleAssignmentPolicy
[36mDEBUG[0m:    [29-12-2023 10:21:42] Starting component SqliteDB
[36mDEBUG[0m:    [29-12-2023 10:21:42] Starting component Posthog
[36mDEBUG[0m:    [29-12-2023 10:21:42] Starting component LocalSegmentManager
[36mDEBUG[0m:    [29-12-2023 10:21:42] Starting component SegmentAPI
[32mINFO[0m:     [29-12-2023 10:21:42] Started server process [[36m19018[0m]
[32mINFO[0m:     [29-12-2023 10:21:42] Waiting for application startup.
[32mINFO[0m:     [29-12-2023 10:21:42] Application startup complete.
[32mINFO[0m:     [29-12-2023 10:21:42] Uvicorn running on [1mhttp://localhost:8000[0m (Press CTRL+C to quit)
[32mINFO[0m:     [29-12-2023 10:21:46] ::1:52024 - "POST /api/v1/collections?tenant=default_tenant&database=default_database HTTP/1.1" 200
[32mINFO[0m:     [29-12-2023 10:21:46] ::1:52025 - "GET /api/v1/collections/movie_reviews?tenant=default_tenant&database=default_database HTTP/1.1" 200
[32mINFO[0m:     [29-12-2023 10:25:10] Shutting down
[32mINFO[0m:     [29-12-2023 10:25:10] Waiting for application shutdown.
[32mINFO[0m:     [29-12-2023 10:25:10] Application shutdown complete.
[32mINFO[0m:     [29-12-2023 10:25:10] Finished server process [[36m19018[0m]
[32mINFO[0m:     [29-12-2023 10:25:17] Set chroma_server_nofile to 65535
[32mINFO[0m:     [29-12-2023 10:25:17] Anonymized telemetry enabled. See                     https://docs.trychroma.com/telemetry for more information.
[36mDEBUG[0m:    [29-12-2023 10:25:17] Starting component System
[36mDEBUG[0m:    [29-12-2023 10:25:17] Starting component OpenTelemetryClient
[36mDEBUG[0m:    [29-12-2023 10:25:17] Starting component SimpleAssignmentPolicy
[36mDEBUG[0m:    [29-12-2023 10:25:17] Starting component SqliteDB
[36mDEBUG[0m:    [29-12-2023 10:25:17] Starting component Posthog
[36mDEBUG[0m:    [29-12-2023 10:25:17] Starting component LocalSegmentManager
[36mDEBUG[0m:    [29-12-2023 10:25:17] Starting component SegmentAPI
[32mINFO[0m:     [29-12-2023 10:25:17] Started server process [[36m19628[0m]
[32mINFO[0m:     [29-12-2023 10:25:17] Waiting for application startup.
[32mINFO[0m:     [29-12-2023 10:25:17] Application startup complete.
[32mINFO[0m:     [29-12-2023 10:25:17] Uvicorn running on [1mhttp://localhost:8000[0m (Press CTRL+C to quit)
[32mINFO[0m:     [29-12-2023 10:25:23] ::1:52049 - "POST /api/v1/collections?tenant=default_tenant&database=default_database HTTP/1.1" 200
[32mINFO[0m:     [29-12-2023 10:25:23] ::1:52050 - "GET /api/v1/collections/movie_reviews?tenant=default_tenant&database=default_database HTTP/1.1" 200
[32mINFO[0m:     [29-12-2023 10:27:48] Shutting down
[32mINFO[0m:     [29-12-2023 10:27:48] Waiting for application shutdown.
[32mINFO[0m:     [29-12-2023 10:27:48] Application shutdown complete.
[32mINFO[0m:     [29-12-2023 10:27:48] Finished server process [[36m19628[0m]
[32mINFO[0m:     [29-12-2023 10:27:50] Set chroma_server_nofile to 65535
[32mINFO[0m:     [29-12-2023 10:27:50] Anonymized telemetry enabled. See                     https://docs.trychroma.com/telemetry for more information.
[36mDEBUG[0m:    [29-12-2023 10:27:50] Starting component System
[36mDEBUG[0m:    [29-12-2023 10:27:50] Starting component OpenTelemetryClient
[36mDEBUG[0m:    [29-12-2023 10:27:50] Starting component SimpleAssignmentPolicy
[36mDEBUG[0m:    [29-12-2023 10:27:50] Starting component SqliteDB
[36mDEBUG[0m:    [29-12-2023 10:27:50] Starting component Posthog
[36mDEBUG[0m:    [29-12-2023 10:27:50] Starting component LocalSegmentManager
[36mDEBUG[0m:    [29-12-2023 10:27:50] Starting component SegmentAPI
[32mINFO[0m:     [29-12-2023 10:27:50] Started server process [[36m20137[0m]
[32mINFO[0m:     [29-12-2023 10:27:50] Waiting for application startup.
[32mINFO[0m:     [29-12-2023 10:27:50] Application startup complete.
[32mINFO[0m:     [29-12-2023 10:27:50] Uvicorn running on [1mhttp://localhost:8000[0m (Press CTRL+C to quit)
[31mERROR[0m:    [29-12-2023 10:27:54] Collection movie_reviews already exists
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/anyio/streams/memory.py", line 94, in receive
    return self.receive_nowait()
  File "/usr/local/lib/python3.9/site-packages/anyio/streams/memory.py", line 89, in receive_nowait
    raise WouldBlock
anyio.WouldBlock

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/base.py", line 78, in call_next
    message = await recv_stream.receive()
  File "/usr/local/lib/python3.9/site-packages/anyio/streams/memory.py", line 114, in receive
    raise EndOfStream
anyio.EndOfStream

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/chromadb/server/fastapi/__init__.py", line 80, in catch_exceptions_middleware
    return await call_next(request)
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/base.py", line 84, in call_next
    raise app_exc
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/base.py", line 70, in coro
    await self.app(scope, receive_or_disconnect, send_no_error)
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/base.py", line 108, in __call__
    response = await self.dispatch_func(request, call_next)
  File "/usr/local/lib/python3.9/site-packages/chromadb/server/fastapi/__init__.py", line 94, in check_http_version_middleware
    return await call_next(request)
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/base.py", line 84, in call_next
    raise app_exc
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/base.py", line 70, in coro
    await self.app(scope, receive_or_disconnect, send_no_error)
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/exceptions.py", line 79, in __call__
    raise exc
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/exceptions.py", line 68, in __call__
    await self.app(scope, receive, sender)
  File "/usr/local/lib/python3.9/site-packages/fastapi/middleware/asyncexitstack.py", line 20, in __call__
    raise e
  File "/usr/local/lib/python3.9/site-packages/fastapi/middleware/asyncexitstack.py", line 17, in __call__
    await self.app(scope, receive, send)
  File "/usr/local/lib/python3.9/site-packages/starlette/routing.py", line 718, in __call__
    await route.handle(scope, receive, send)
  File "/usr/local/lib/python3.9/site-packages/starlette/routing.py", line 276, in handle
    await self.app(scope, receive, send)
  File "/usr/local/lib/python3.9/site-packages/starlette/routing.py", line 66, in app
    response = await func(request)
  File "/usr/local/lib/python3.9/site-packages/fastapi/routing.py", line 273, in app
    raw_response = await run_endpoint_function(
  File "/usr/local/lib/python3.9/site-packages/fastapi/routing.py", line 192, in run_endpoint_function
    return await run_in_threadpool(dependant.call, **values)
  File "/usr/local/lib/python3.9/site-packages/starlette/concurrency.py", line 41, in run_in_threadpool
    return await anyio.to_thread.run_sync(func, *args)
  File "/usr/local/lib/python3.9/site-packages/anyio/to_thread.py", line 31, in run_sync
    return await get_asynclib().run_sync_in_worker_thread(
  File "/usr/local/lib/python3.9/site-packages/anyio/_backends/_asyncio.py", line 937, in run_sync_in_worker_thread
    return await future
  File "/usr/local/lib/python3.9/site-packages/anyio/_backends/_asyncio.py", line 867, in run
    result = context.run(func, *args)
  File "/usr/local/lib/python3.9/site-packages/chromadb/telemetry/opentelemetry/__init__.py", line 127, in wrapper
    return f(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/chromadb/auth/fastapi.py", line 234, in wrapped
    return f(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/chromadb/server/fastapi/__init__.py", line 408, in create_collection
    return self._api.create_collection(
  File "/usr/local/lib/python3.9/site-packages/chromadb/telemetry/opentelemetry/__init__.py", line 127, in wrapper
    return f(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/chromadb/api/segment.py", line 166, in create_collection
    coll, created = self._sysdb.create_collection(
  File "/usr/local/lib/python3.9/site-packages/chromadb/telemetry/opentelemetry/__init__.py", line 127, in wrapper
    return f(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/chromadb/db/mixins/sysdb.py", line 225, in create_collection
    raise UniqueConstraintError(f"Collection {name} already exists")
chromadb.db.base.UniqueConstraintError: Collection movie_reviews already exists
[32mINFO[0m:     [29-12-2023 10:27:54] ::1:52078 - "POST /api/v1/collections?tenant=default_tenant&database=default_database HTTP/1.1" 500
[32mINFO[0m:     [29-12-2023 10:28:06] Shutting down
[32mINFO[0m:     [29-12-2023 10:28:06] Waiting for application shutdown.
[32mINFO[0m:     [29-12-2023 10:28:06] Application shutdown complete.
[32mINFO[0m:     [29-12-2023 10:28:06] Finished server process [[36m20137[0m]
[32mINFO[0m:     [29-12-2023 10:28:08] Set chroma_server_nofile to 65535
[32mINFO[0m:     [29-12-2023 10:28:08] Anonymized telemetry enabled. See                     https://docs.trychroma.com/telemetry for more information.
[36mDEBUG[0m:    [29-12-2023 10:28:08] Starting component System
[36mDEBUG[0m:    [29-12-2023 10:28:08] Starting component OpenTelemetryClient
[36mDEBUG[0m:    [29-12-2023 10:28:08] Starting component SimpleAssignmentPolicy
[36mDEBUG[0m:    [29-12-2023 10:28:08] Starting component SqliteDB
[36mDEBUG[0m:    [29-12-2023 10:28:08] Starting component Posthog
[36mDEBUG[0m:    [29-12-2023 10:28:08] Starting component LocalSegmentManager
[36mDEBUG[0m:    [29-12-2023 10:28:08] Starting component SegmentAPI
[32mINFO[0m:     [29-12-2023 10:28:08] Started server process [[36m20196[0m]
[32mINFO[0m:     [29-12-2023 10:28:08] Waiting for application startup.
[32mINFO[0m:     [29-12-2023 10:28:08] Application startup complete.
[32mINFO[0m:     [29-12-2023 10:28:08] Uvicorn running on [1mhttp://localhost:8000[0m (Press CTRL+C to quit)
[32mINFO[0m:     [29-12-2023 10:28:11] ::1:52080 - "POST /api/v1/collections?tenant=default_tenant&database=default_database HTTP/1.1" 200
[32mINFO[0m:     [29-12-2023 10:28:11] ::1:52081 - "GET /api/v1/collections/movie_reviews?tenant=default_tenant&database=default_database HTTP/1.1" 200
[32mINFO[0m:     [29-12-2023 10:30:52] Shutting down
[32mINFO[0m:     [29-12-2023 10:30:52] Waiting for application shutdown.
[32mINFO[0m:     [29-12-2023 10:30:52] Application shutdown complete.
[32mINFO[0m:     [29-12-2023 10:30:52] Finished server process [[36m20196[0m]
[32mINFO[0m:     [29-12-2023 10:30:59] Set chroma_server_nofile to 65535
[32mINFO[0m:     [29-12-2023 10:30:59] Anonymized telemetry enabled. See                     https://docs.trychroma.com/telemetry for more information.
[36mDEBUG[0m:    [29-12-2023 10:30:59] Starting component System
[36mDEBUG[0m:    [29-12-2023 10:30:59] Starting component OpenTelemetryClient
[36mDEBUG[0m:    [29-12-2023 10:30:59] Starting component SimpleAssignmentPolicy
[36mDEBUG[0m:    [29-12-2023 10:30:59] Starting component SqliteDB
[36mDEBUG[0m:    [29-12-2023 10:30:59] Starting component Posthog
[36mDEBUG[0m:    [29-12-2023 10:30:59] Starting component LocalSegmentManager
[36mDEBUG[0m:    [29-12-2023 10:30:59] Starting component SegmentAPI
[32mINFO[0m:     [29-12-2023 10:30:59] Started server process [[36m20656[0m]
[32mINFO[0m:     [29-12-2023 10:30:59] Waiting for application startup.
[32mINFO[0m:     [29-12-2023 10:30:59] Application startup complete.
[32mINFO[0m:     [29-12-2023 10:30:59] Uvicorn running on [1mhttp://localhost:8000[0m (Press CTRL+C to quit)
[32mINFO[0m:     [29-12-2023 10:41:43] Shutting down
[32mINFO[0m:     [29-12-2023 10:41:43] Waiting for application shutdown.
[32mINFO[0m:     [29-12-2023 10:41:43] Application shutdown complete.
[32mINFO[0m:     [29-12-2023 10:41:43] Finished server process [[36m20656[0m]
[32mINFO[0m:     [29-12-2023 10:41:50] Set chroma_server_nofile to 65535
[32mINFO[0m:     [29-12-2023 10:41:50] Anonymized telemetry enabled. See                     https://docs.trychroma.com/telemetry for more information.
[36mDEBUG[0m:    [29-12-2023 10:41:50] Starting component System
[36mDEBUG[0m:    [29-12-2023 10:41:50] Starting component OpenTelemetryClient
[36mDEBUG[0m:    [29-12-2023 10:41:50] Starting component SimpleAssignmentPolicy
[36mDEBUG[0m:    [29-12-2023 10:41:50] Starting component SqliteDB
[36mDEBUG[0m:    [29-12-2023 10:41:50] Starting component Posthog
[36mDEBUG[0m:    [29-12-2023 10:41:50] Starting component LocalSegmentManager
[36mDEBUG[0m:    [29-12-2023 10:41:50] Starting component SegmentAPI
[32mINFO[0m:     [29-12-2023 10:41:50] Started server process [[36m22532[0m]
[32mINFO[0m:     [29-12-2023 10:41:50] Waiting for application startup.
[32mINFO[0m:     [29-12-2023 10:41:50] Application startup complete.
[32mINFO[0m:     [29-12-2023 10:41:50] Uvicorn running on [1mhttp://localhost:8000[0m (Press CTRL+C to quit)
[32mINFO[0m:     [29-12-2023 10:43:40] Shutting down
[32mINFO[0m:     [29-12-2023 10:43:40] Waiting for application shutdown.
[32mINFO[0m:     [29-12-2023 10:43:40] Application shutdown complete.
[32mINFO[0m:     [29-12-2023 10:43:40] Finished server process [[36m22532[0m]
[32mINFO[0m:     [29-12-2023 10:43:41] Set chroma_server_nofile to 65535
[32mINFO[0m:     [29-12-2023 10:43:41] Anonymized telemetry enabled. See                     https://docs.trychroma.com/telemetry for more information.
[36mDEBUG[0m:    [29-12-2023 10:43:41] Starting component System
[36mDEBUG[0m:    [29-12-2023 10:43:41] Starting component OpenTelemetryClient
[36mDEBUG[0m:    [29-12-2023 10:43:41] Starting component SimpleAssignmentPolicy
[36mDEBUG[0m:    [29-12-2023 10:43:41] Starting component SqliteDB
[36mDEBUG[0m:    [29-12-2023 10:43:41] Starting component Posthog
[36mDEBUG[0m:    [29-12-2023 10:43:41] Starting component LocalSegmentManager
[36mDEBUG[0m:    [29-12-2023 10:43:41] Starting component SegmentAPI
[32mINFO[0m:     [29-12-2023 10:43:41] Started server process [[36m22844[0m]
[32mINFO[0m:     [29-12-2023 10:43:41] Waiting for application startup.
[32mINFO[0m:     [29-12-2023 10:43:41] Application startup complete.
[32mINFO[0m:     [29-12-2023 10:43:41] Uvicorn running on [1mhttp://localhost:8000[0m (Press CTRL+C to quit)
[32mINFO[0m:     [29-12-2023 10:43:44] ::1:52181 - "POST /api/v1/collections?tenant=default_tenant&database=default_database HTTP/1.1" 200
[32mINFO[0m:     [29-12-2023 10:43:44] ::1:52182 - "GET /api/v1/collections/movie_reviews?tenant=default_tenant&database=default_database HTTP/1.1" 200
[32mINFO[0m:     [29-12-2023 10:46:35] Shutting down
[32mINFO[0m:     [29-12-2023 10:46:35] Waiting for application shutdown.
[32mINFO[0m:     [29-12-2023 10:46:35] Application shutdown complete.
[32mINFO[0m:     [29-12-2023 10:46:35] Finished server process [[36m22844[0m]
[32mINFO[0m:     [29-12-2023 10:46:48] Set chroma_server_nofile to 65535
[32mINFO[0m:     [29-12-2023 10:46:49] Anonymized telemetry enabled. See                     https://docs.trychroma.com/telemetry for more information.
[36mDEBUG[0m:    [29-12-2023 10:46:49] Starting component System
[36mDEBUG[0m:    [29-12-2023 10:46:49] Starting component OpenTelemetryClient
[36mDEBUG[0m:    [29-12-2023 10:46:49] Starting component SimpleAssignmentPolicy
[36mDEBUG[0m:    [29-12-2023 10:46:49] Starting component SqliteDB
[36mDEBUG[0m:    [29-12-2023 10:46:49] Starting component Posthog
[36mDEBUG[0m:    [29-12-2023 10:46:49] Starting component LocalSegmentManager
[36mDEBUG[0m:    [29-12-2023 10:46:49] Starting component SegmentAPI
[32mINFO[0m:     [29-12-2023 10:46:49] Started server process [[36m23363[0m]
[32mINFO[0m:     [29-12-2023 10:46:49] Waiting for application startup.
[32mINFO[0m:     [29-12-2023 10:46:49] Application startup complete.
[32mINFO[0m:     [29-12-2023 10:46:49] Uvicorn running on [1mhttp://localhost:8000[0m (Press CTRL+C to quit)
[32mINFO[0m:     [29-12-2023 11:01:25] Shutting down
[32mINFO[0m:     [29-12-2023 11:01:25] Waiting for application shutdown.
[32mINFO[0m:     [29-12-2023 11:01:25] Application shutdown complete.
[32mINFO[0m:     [29-12-2023 11:01:25] Finished server process [[36m23363[0m]
[32mINFO[0m:     [29-12-2023 11:01:31] Set chroma_server_nofile to 65535
[32mINFO[0m:     [29-12-2023 11:01:31] Anonymized telemetry enabled. See                     https://docs.trychroma.com/telemetry for more information.
[36mDEBUG[0m:    [29-12-2023 11:01:31] Starting component System
[36mDEBUG[0m:    [29-12-2023 11:01:31] Starting component OpenTelemetryClient
[36mDEBUG[0m:    [29-12-2023 11:01:31] Starting component SimpleAssignmentPolicy
[36mDEBUG[0m:    [29-12-2023 11:01:31] Starting component SqliteDB
[36mDEBUG[0m:    [29-12-2023 11:01:31] Starting component Posthog
[36mDEBUG[0m:    [29-12-2023 11:01:31] Starting component LocalSegmentManager
[36mDEBUG[0m:    [29-12-2023 11:01:31] Starting component SegmentAPI
[32mINFO[0m:     [29-12-2023 11:01:31] Started server process [[36m26397[0m]
[32mINFO[0m:     [29-12-2023 11:01:31] Waiting for application startup.
[32mINFO[0m:     [29-12-2023 11:01:31] Application startup complete.
[32mINFO[0m:     [29-12-2023 11:01:31] Uvicorn running on [1mhttp://localhost:8000[0m (Press CTRL+C to quit)
[32mINFO[0m:     [29-12-2023 11:16:29] Shutting down
[32mINFO[0m:     [29-12-2023 11:16:29] Waiting for application shutdown.
[32mINFO[0m:     [29-12-2023 11:16:29] Application shutdown complete.
[32mINFO[0m:     [29-12-2023 11:16:29] Finished server process [[36m26397[0m]
[32mINFO[0m:     [29-12-2023 11:16:31] Set chroma_server_nofile to 65535
[32mINFO[0m:     [29-12-2023 11:16:31] Anonymized telemetry enabled. See                     https://docs.trychroma.com/telemetry for more information.
[36mDEBUG[0m:    [29-12-2023 11:16:31] Starting component System
[36mDEBUG[0m:    [29-12-2023 11:16:31] Starting component OpenTelemetryClient
[36mDEBUG[0m:    [29-12-2023 11:16:31] Starting component SimpleAssignmentPolicy
[36mDEBUG[0m:    [29-12-2023 11:16:31] Starting component SqliteDB
[36mDEBUG[0m:    [29-12-2023 11:16:31] Starting component Posthog
[36mDEBUG[0m:    [29-12-2023 11:16:31] Starting component LocalSegmentManager
[36mDEBUG[0m:    [29-12-2023 11:16:31] Starting component SegmentAPI
[32mINFO[0m:     [29-12-2023 11:16:31] Started server process [[36m29293[0m]
[32mINFO[0m:     [29-12-2023 11:16:31] Waiting for application startup.
[32mINFO[0m:     [29-12-2023 11:16:31] Application startup complete.
[32mINFO[0m:     [29-12-2023 11:16:31] Uvicorn running on [1mhttp://localhost:8000[0m (Press CTRL+C to quit)
[32mINFO[0m:     [29-12-2023 11:16:42] Shutting down
[32mINFO[0m:     [29-12-2023 11:16:42] Waiting for application shutdown.
[32mINFO[0m:     [29-12-2023 11:16:42] Application shutdown complete.
[32mINFO[0m:     [29-12-2023 11:16:42] Finished server process [[36m29293[0m]
[32mINFO[0m:     [29-12-2023 11:16:59] Set chroma_server_nofile to 65535
[32mINFO[0m:     [29-12-2023 11:17:00] Anonymized telemetry enabled. See                     https://docs.trychroma.com/telemetry for more information.
[36mDEBUG[0m:    [29-12-2023 11:17:00] Starting component System
[36mDEBUG[0m:    [29-12-2023 11:17:00] Starting component OpenTelemetryClient
[36mDEBUG[0m:    [29-12-2023 11:17:00] Starting component SimpleAssignmentPolicy
[36mDEBUG[0m:    [29-12-2023 11:17:00] Starting component SqliteDB
[36mDEBUG[0m:    [29-12-2023 11:17:00] Starting component Posthog
[36mDEBUG[0m:    [29-12-2023 11:17:00] Starting component LocalSegmentManager
[36mDEBUG[0m:    [29-12-2023 11:17:00] Starting component SegmentAPI
[32mINFO[0m:     [29-12-2023 11:17:00] Started server process [[36m29377[0m]
[32mINFO[0m:     [29-12-2023 11:17:00] Waiting for application startup.
[32mINFO[0m:     [29-12-2023 11:17:00] Application startup complete.
[32mINFO[0m:     [29-12-2023 11:17:00] Uvicorn running on [1mhttp://localhost:8000[0m (Press CTRL+C to quit)
[32mINFO[0m:     [29-12-2023 11:17:17] Shutting down
[32mINFO[0m:     [29-12-2023 11:17:17] Waiting for application shutdown.
[32mINFO[0m:     [29-12-2023 11:17:17] Application shutdown complete.
[32mINFO[0m:     [29-12-2023 11:17:17] Finished server process [[36m29377[0m]
[32mINFO[0m:     [29-12-2023 11:17:19] Set chroma_server_nofile to 65535
[32mINFO[0m:     [29-12-2023 11:17:19] Anonymized telemetry enabled. See                     https://docs.trychroma.com/telemetry for more information.
[36mDEBUG[0m:    [29-12-2023 11:17:19] Starting component System
[36mDEBUG[0m:    [29-12-2023 11:17:19] Starting component OpenTelemetryClient
[36mDEBUG[0m:    [29-12-2023 11:17:19] Starting component SimpleAssignmentPolicy
[36mDEBUG[0m:    [29-12-2023 11:17:19] Starting component SqliteDB
[36mDEBUG[0m:    [29-12-2023 11:17:19] Starting component Posthog
[36mDEBUG[0m:    [29-12-2023 11:17:19] Starting component LocalSegmentManager
[36mDEBUG[0m:    [29-12-2023 11:17:19] Starting component SegmentAPI
[32mINFO[0m:     [29-12-2023 11:17:19] Started server process [[36m29431[0m]
[32mINFO[0m:     [29-12-2023 11:17:19] Waiting for application startup.
[32mINFO[0m:     [29-12-2023 11:17:19] Application startup complete.
[32mINFO[0m:     [29-12-2023 11:17:19] Uvicorn running on [1mhttp://localhost:8000[0m (Press CTRL+C to quit)
[32mINFO[0m:     [29-12-2023 11:17:48] Shutting down
[32mINFO[0m:     [29-12-2023 11:17:48] Waiting for application shutdown.
[32mINFO[0m:     [29-12-2023 11:17:48] Application shutdown complete.
[32mINFO[0m:     [29-12-2023 11:17:48] Finished server process [[36m29431[0m]
[32mINFO[0m:     [29-12-2023 11:17:49] Set chroma_server_nofile to 65535
[32mINFO[0m:     [29-12-2023 11:17:49] Anonymized telemetry enabled. See                     https://docs.trychroma.com/telemetry for more information.
[36mDEBUG[0m:    [29-12-2023 11:17:49] Starting component System
[36mDEBUG[0m:    [29-12-2023 11:17:49] Starting component OpenTelemetryClient
[36mDEBUG[0m:    [29-12-2023 11:17:49] Starting component SimpleAssignmentPolicy
[36mDEBUG[0m:    [29-12-2023 11:17:49] Starting component SqliteDB
[36mDEBUG[0m:    [29-12-2023 11:17:49] Starting component Posthog
[36mDEBUG[0m:    [29-12-2023 11:17:49] Starting component LocalSegmentManager
[36mDEBUG[0m:    [29-12-2023 11:17:49] Starting component SegmentAPI
[32mINFO[0m:     [29-12-2023 11:17:49] Started server process [[36m29526[0m]
[32mINFO[0m:     [29-12-2023 11:17:49] Waiting for application startup.
[32mINFO[0m:     [29-12-2023 11:17:49] Application startup complete.
[32mINFO[0m:     [29-12-2023 11:17:49] Uvicorn running on [1mhttp://localhost:8000[0m (Press CTRL+C to quit)
[32mINFO[0m:     [29-12-2023 11:18:52] Shutting down
[32mINFO[0m:     [29-12-2023 11:18:52] Waiting for application shutdown.
[32mINFO[0m:     [29-12-2023 11:18:52] Application shutdown complete.
[32mINFO[0m:     [29-12-2023 11:18:52] Finished server process [[36m29526[0m]
[32mINFO[0m:     [29-12-2023 11:18:53] Set chroma_server_nofile to 65535
[32mINFO[0m:     [29-12-2023 11:18:53] Anonymized telemetry enabled. See                     https://docs.trychroma.com/telemetry for more information.
[36mDEBUG[0m:    [29-12-2023 11:18:53] Starting component System
[36mDEBUG[0m:    [29-12-2023 11:18:53] Starting component OpenTelemetryClient
[36mDEBUG[0m:    [29-12-2023 11:18:53] Starting component SimpleAssignmentPolicy
[36mDEBUG[0m:    [29-12-2023 11:18:53] Starting component SqliteDB
[36mDEBUG[0m:    [29-12-2023 11:18:53] Starting component Posthog
[36mDEBUG[0m:    [29-12-2023 11:18:53] Starting component LocalSegmentManager
[36mDEBUG[0m:    [29-12-2023 11:18:53] Starting component SegmentAPI
[32mINFO[0m:     [29-12-2023 11:18:53] Started server process [[36m29725[0m]
[32mINFO[0m:     [29-12-2023 11:18:53] Waiting for application startup.
[32mINFO[0m:     [29-12-2023 11:18:53] Application startup complete.
[32mINFO[0m:     [29-12-2023 11:18:53] Uvicorn running on [1mhttp://localhost:8000[0m (Press CTRL+C to quit)
[32mINFO[0m:     [29-12-2023 11:18:55] ::1:52445 - "POST /api/v1/collections?tenant=default_tenant&database=default_database HTTP/1.1" 200
[32mINFO[0m:     [29-12-2023 11:18:55] ::1:52446 - "GET /api/v1/collections?tenant=default_tenant&database=default_database HTTP/1.1" 200
[32mINFO[0m:     [29-12-2023 11:18:55] ::1:52445 - "GET /api/v1/collections/movie_reviews?tenant=default_tenant&database=default_database HTTP/1.1" 200
[32mINFO[0m:     [29-12-2023 11:31:56] Shutting down
[32mINFO[0m:     [29-12-2023 11:31:57] Waiting for application shutdown.
[32mINFO[0m:     [29-12-2023 11:31:57] Application shutdown complete.
[32mINFO[0m:     [29-12-2023 11:31:57] Finished server process [[36m29725[0m]
[32mINFO[0m:     [29-12-2023 11:32:06] Set chroma_server_nofile to 65535
[32mINFO[0m:     [29-12-2023 11:32:06] Anonymized telemetry enabled. See                     https://docs.trychroma.com/telemetry for more information.
[36mDEBUG[0m:    [29-12-2023 11:32:06] Starting component System
[36mDEBUG[0m:    [29-12-2023 11:32:06] Starting component OpenTelemetryClient
[36mDEBUG[0m:    [29-12-2023 11:32:06] Starting component SimpleAssignmentPolicy
[36mDEBUG[0m:    [29-12-2023 11:32:06] Starting component SqliteDB
[36mDEBUG[0m:    [29-12-2023 11:32:06] Starting component Posthog
[36mDEBUG[0m:    [29-12-2023 11:32:06] Starting component LocalSegmentManager
[36mDEBUG[0m:    [29-12-2023 11:32:06] Starting component SegmentAPI
[32mINFO[0m:     [29-12-2023 11:32:06] Started server process [[36m32327[0m]
[32mINFO[0m:     [29-12-2023 11:32:06] Waiting for application startup.
[32mINFO[0m:     [29-12-2023 11:32:06] Application startup complete.
[32mINFO[0m:     [29-12-2023 11:32:06] Uvicorn running on [1mhttp://localhost:8000[0m (Press CTRL+C to quit)
[32mINFO[0m:     [29-12-2023 11:32:09] ::1:52579 - "POST /api/v1/collections?tenant=default_tenant&database=default_database HTTP/1.1" 200
[32mINFO[0m:     [29-12-2023 11:32:09] ::1:52580 - "GET /api/v1/collections?tenant=default_tenant&database=default_database HTTP/1.1" 200
[32mINFO[0m:     [29-12-2023 11:32:09] ::1:52579 - "GET /api/v1/collections/movie_reviews?tenant=default_tenant&database=default_database HTTP/1.1" 200
[32mINFO[0m:     [29-12-2023 12:03:25] Shutting down
[32mINFO[0m:     [29-12-2023 12:03:25] Waiting for application shutdown.
[32mINFO[0m:     [29-12-2023 12:03:25] Application shutdown complete.
[32mINFO[0m:     [29-12-2023 12:03:25] Finished server process [[36m32327[0m]
[32mINFO[0m:     [29-12-2023 12:03:32] Set chroma_server_nofile to 65535
[32mINFO[0m:     [29-12-2023 12:03:32] Anonymized telemetry enabled. See                     https://docs.trychroma.com/telemetry for more information.
[36mDEBUG[0m:    [29-12-2023 12:03:32] Starting component System
[36mDEBUG[0m:    [29-12-2023 12:03:32] Starting component OpenTelemetryClient
[36mDEBUG[0m:    [29-12-2023 12:03:32] Starting component SimpleAssignmentPolicy
[36mDEBUG[0m:    [29-12-2023 12:03:32] Starting component SqliteDB
[36mDEBUG[0m:    [29-12-2023 12:03:32] Starting component Posthog
[36mDEBUG[0m:    [29-12-2023 12:03:32] Starting component LocalSegmentManager
[36mDEBUG[0m:    [29-12-2023 12:03:32] Starting component SegmentAPI
[32mINFO[0m:     [29-12-2023 12:03:32] Started server process [[36m37947[0m]
[32mINFO[0m:     [29-12-2023 12:03:32] Waiting for application startup.
[32mINFO[0m:     [29-12-2023 12:03:32] Application startup complete.
[32mINFO[0m:     [29-12-2023 12:03:32] Uvicorn running on [1mhttp://localhost:8000[0m (Press CTRL+C to quit)
[32mINFO[0m:     [29-12-2023 12:03:36] ::1:52806 - "POST /api/v1/collections?tenant=default_tenant&database=default_database HTTP/1.1" 200
[32mINFO[0m:     [29-12-2023 12:03:36] ::1:52807 - "GET /api/v1/collections?tenant=default_tenant&database=default_database HTTP/1.1" 200
[32mINFO[0m:     [29-12-2023 12:03:36] ::1:52806 - "GET /api/v1/collections/movie_reviews?tenant=default_tenant&database=default_database HTTP/1.1" 200
[32mINFO[0m:     [29-12-2023 13:01:58] Shutting down
[32mINFO[0m:     [29-12-2023 13:01:58] Waiting for application shutdown.
[32mINFO[0m:     [29-12-2023 13:01:58] Application shutdown complete.
[32mINFO[0m:     [29-12-2023 13:01:58] Finished server process [[36m37947[0m]
[32mINFO[0m:     [29-12-2023 13:02:07] Set chroma_server_nofile to 65535
[32mINFO[0m:     [29-12-2023 13:02:07] Anonymized telemetry enabled. See                     https://docs.trychroma.com/telemetry for more information.
[36mDEBUG[0m:    [29-12-2023 13:02:07] Starting component System
[36mDEBUG[0m:    [29-12-2023 13:02:07] Starting component OpenTelemetryClient
[36mDEBUG[0m:    [29-12-2023 13:02:07] Starting component SimpleAssignmentPolicy
[36mDEBUG[0m:    [29-12-2023 13:02:07] Starting component SqliteDB
[36mDEBUG[0m:    [29-12-2023 13:02:07] Starting component Posthog
[36mDEBUG[0m:    [29-12-2023 13:02:07] Starting component LocalSegmentManager
[36mDEBUG[0m:    [29-12-2023 13:02:07] Starting component SegmentAPI
[32mINFO[0m:     [29-12-2023 13:02:07] Started server process [[36m48107[0m]
[32mINFO[0m:     [29-12-2023 13:02:07] Waiting for application startup.
[32mINFO[0m:     [29-12-2023 13:02:07] Application startup complete.
[32mINFO[0m:     [29-12-2023 13:02:07] Uvicorn running on [1mhttp://localhost:8000[0m (Press CTRL+C to quit)
[31mERROR[0m:    [29-12-2023 13:02:09] Collection movie_reviews does not exist.
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/anyio/streams/memory.py", line 94, in receive
    return self.receive_nowait()
  File "/usr/local/lib/python3.9/site-packages/anyio/streams/memory.py", line 89, in receive_nowait
    raise WouldBlock
anyio.WouldBlock

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/base.py", line 78, in call_next
    message = await recv_stream.receive()
  File "/usr/local/lib/python3.9/site-packages/anyio/streams/memory.py", line 114, in receive
    raise EndOfStream
anyio.EndOfStream

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/chromadb/server/fastapi/__init__.py", line 80, in catch_exceptions_middleware
    return await call_next(request)
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/base.py", line 84, in call_next
    raise app_exc
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/base.py", line 70, in coro
    await self.app(scope, receive_or_disconnect, send_no_error)
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/base.py", line 108, in __call__
    response = await self.dispatch_func(request, call_next)
  File "/usr/local/lib/python3.9/site-packages/chromadb/server/fastapi/__init__.py", line 94, in check_http_version_middleware
    return await call_next(request)
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/base.py", line 84, in call_next
    raise app_exc
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/base.py", line 70, in coro
    await self.app(scope, receive_or_disconnect, send_no_error)
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/exceptions.py", line 79, in __call__
    raise exc
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/exceptions.py", line 68, in __call__
    await self.app(scope, receive, sender)
  File "/usr/local/lib/python3.9/site-packages/fastapi/middleware/asyncexitstack.py", line 20, in __call__
    raise e
  File "/usr/local/lib/python3.9/site-packages/fastapi/middleware/asyncexitstack.py", line 17, in __call__
    await self.app(scope, receive, send)
  File "/usr/local/lib/python3.9/site-packages/starlette/routing.py", line 718, in __call__
    await route.handle(scope, receive, send)
  File "/usr/local/lib/python3.9/site-packages/starlette/routing.py", line 276, in handle
    await self.app(scope, receive, send)
  File "/usr/local/lib/python3.9/site-packages/starlette/routing.py", line 66, in app
    response = await func(request)
  File "/usr/local/lib/python3.9/site-packages/fastapi/routing.py", line 273, in app
    raw_response = await run_endpoint_function(
  File "/usr/local/lib/python3.9/site-packages/fastapi/routing.py", line 192, in run_endpoint_function
    return await run_in_threadpool(dependant.call, **values)
  File "/usr/local/lib/python3.9/site-packages/starlette/concurrency.py", line 41, in run_in_threadpool
    return await anyio.to_thread.run_sync(func, *args)
  File "/usr/local/lib/python3.9/site-packages/anyio/to_thread.py", line 31, in run_sync
    return await get_asynclib().run_sync_in_worker_thread(
  File "/usr/local/lib/python3.9/site-packages/anyio/_backends/_asyncio.py", line 937, in run_sync_in_worker_thread
    return await future
  File "/usr/local/lib/python3.9/site-packages/anyio/_backends/_asyncio.py", line 867, in run
    result = context.run(func, *args)
  File "/usr/local/lib/python3.9/site-packages/chromadb/telemetry/opentelemetry/__init__.py", line 127, in wrapper
    return f(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/chromadb/auth/fastapi.py", line 234, in wrapped
    return f(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/chromadb/server/fastapi/__init__.py", line 433, in get_collection
    return self._api.get_collection(
  File "/usr/local/lib/python3.9/site-packages/chromadb/telemetry/opentelemetry/__init__.py", line 127, in wrapper
    return f(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/chromadb/api/segment.py", line 260, in get_collection
    raise ValueError(f"Collection {name} does not exist.")
ValueError: Collection movie_reviews does not exist.
[32mINFO[0m:     [29-12-2023 13:02:09] ::1:53126 - "GET /api/v1/collections/movie_reviews?tenant=default_tenant&database=default_database HTTP/1.1" 500
[32mINFO[0m:     [29-12-2023 13:03:58] Shutting down
[32mINFO[0m:     [29-12-2023 13:03:58] Waiting for application shutdown.
[32mINFO[0m:     [29-12-2023 13:03:58] Application shutdown complete.
[32mINFO[0m:     [29-12-2023 13:03:58] Finished server process [[36m48107[0m]
[32mINFO[0m:     [29-12-2023 13:04:06] Set chroma_server_nofile to 65535
[32mINFO[0m:     [29-12-2023 13:04:06] Anonymized telemetry enabled. See                     https://docs.trychroma.com/telemetry for more information.
[36mDEBUG[0m:    [29-12-2023 13:04:06] Starting component System
[36mDEBUG[0m:    [29-12-2023 13:04:06] Starting component OpenTelemetryClient
[36mDEBUG[0m:    [29-12-2023 13:04:06] Starting component SimpleAssignmentPolicy
[36mDEBUG[0m:    [29-12-2023 13:04:06] Starting component SqliteDB
[36mDEBUG[0m:    [29-12-2023 13:04:06] Starting component Posthog
[36mDEBUG[0m:    [29-12-2023 13:04:06] Starting component LocalSegmentManager
[36mDEBUG[0m:    [29-12-2023 13:04:06] Starting component SegmentAPI
[32mINFO[0m:     [29-12-2023 13:04:06] Started server process [[36m48448[0m]
[32mINFO[0m:     [29-12-2023 13:04:06] Waiting for application startup.
[32mINFO[0m:     [29-12-2023 13:04:06] Application startup complete.
[32mINFO[0m:     [29-12-2023 13:04:06] Uvicorn running on [1mhttp://localhost:8000[0m (Press CTRL+C to quit)
[32mINFO[0m:     [29-12-2023 13:04:09] ::1:53132 - "POST /api/v1/collections?tenant=default_tenant&database=default_database HTTP/1.1" 200
[32mINFO[0m:     [29-12-2023 13:04:09] ::1:53133 - "GET /api/v1/collections?tenant=default_tenant&database=default_database HTTP/1.1" 200
[32mINFO[0m:     [29-12-2023 13:04:09] ::1:53132 - "GET /api/v1/collections/movie_reviews?tenant=default_tenant&database=default_database HTTP/1.1" 200
[32mINFO[0m:     [29-12-2023 13:07:29] Shutting down
[32mINFO[0m:     [29-12-2023 13:07:29] Waiting for application shutdown.
[32mINFO[0m:     [29-12-2023 13:07:29] Application shutdown complete.
[32mINFO[0m:     [29-12-2023 13:07:29] Finished server process [[36m48448[0m]
[32mINFO[0m:     [29-12-2023 13:07:37] Set chroma_server_nofile to 65535
[32mINFO[0m:     [29-12-2023 13:07:37] Anonymized telemetry enabled. See                     https://docs.trychroma.com/telemetry for more information.
[36mDEBUG[0m:    [29-12-2023 13:07:37] Starting component System
[36mDEBUG[0m:    [29-12-2023 13:07:37] Starting component OpenTelemetryClient
[36mDEBUG[0m:    [29-12-2023 13:07:37] Starting component SimpleAssignmentPolicy
[36mDEBUG[0m:    [29-12-2023 13:07:37] Starting component SqliteDB
[36mDEBUG[0m:    [29-12-2023 13:07:37] Starting component Posthog
[36mDEBUG[0m:    [29-12-2023 13:07:37] Starting component LocalSegmentManager
[36mDEBUG[0m:    [29-12-2023 13:07:37] Starting component SegmentAPI
[32mINFO[0m:     [29-12-2023 13:07:37] Started server process [[36m49038[0m]
[32mINFO[0m:     [29-12-2023 13:07:37] Waiting for application startup.
[32mINFO[0m:     [29-12-2023 13:07:37] Application startup complete.
[32mINFO[0m:     [29-12-2023 13:07:37] Uvicorn running on [1mhttp://localhost:8000[0m (Press CTRL+C to quit)
[32mINFO[0m:     [29-12-2023 13:07:39] ::1:53145 - "POST /api/v1/collections?tenant=default_tenant&database=default_database HTTP/1.1" 200
[32mINFO[0m:     [29-12-2023 13:07:39] ::1:53146 - "GET /api/v1/collections?tenant=default_tenant&database=default_database HTTP/1.1" 200
[32mINFO[0m:     [29-12-2023 13:07:39] ::1:53145 - "GET /api/v1/collections/movie_reviews?tenant=default_tenant&database=default_database HTTP/1.1" 200
[32mINFO[0m:     [29-12-2023 13:12:26] ::1:53165 - "GET /api/v1/collections/movie_reviews?tenant=default_tenant&database=default_database HTTP/1.1" 200
[32mINFO[0m:     [29-12-2023 13:12:37] Shutting down
[32mINFO[0m:     [29-12-2023 13:12:37] Waiting for application shutdown.
[32mINFO[0m:     [29-12-2023 13:12:37] Application shutdown complete.
[32mINFO[0m:     [29-12-2023 13:12:37] Finished server process [[36m49038[0m]
[32mINFO[0m:     [29-12-2023 13:12:39] Set chroma_server_nofile to 65535
[32mINFO[0m:     [29-12-2023 13:12:39] Anonymized telemetry enabled. See                     https://docs.trychroma.com/telemetry for more information.
[36mDEBUG[0m:    [29-12-2023 13:12:39] Starting component System
[36mDEBUG[0m:    [29-12-2023 13:12:39] Starting component OpenTelemetryClient
[36mDEBUG[0m:    [29-12-2023 13:12:39] Starting component SimpleAssignmentPolicy
[36mDEBUG[0m:    [29-12-2023 13:12:39] Starting component SqliteDB
[36mDEBUG[0m:    [29-12-2023 13:12:39] Starting component Posthog
[36mDEBUG[0m:    [29-12-2023 13:12:39] Starting component LocalSegmentManager
[36mDEBUG[0m:    [29-12-2023 13:12:39] Starting component SegmentAPI
[32mINFO[0m:     [29-12-2023 13:12:39] Started server process [[36m49875[0m]
[32mINFO[0m:     [29-12-2023 13:12:39] Waiting for application startup.
[32mINFO[0m:     [29-12-2023 13:12:39] Application startup complete.
[32mINFO[0m:     [29-12-2023 13:12:39] Uvicorn running on [1mhttp://localhost:8000[0m (Press CTRL+C to quit)
[31mERROR[0m:    [29-12-2023 13:12:42] Collection movie_reviews does not exist.
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/anyio/streams/memory.py", line 94, in receive
    return self.receive_nowait()
  File "/usr/local/lib/python3.9/site-packages/anyio/streams/memory.py", line 89, in receive_nowait
    raise WouldBlock
anyio.WouldBlock

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/base.py", line 78, in call_next
    message = await recv_stream.receive()
  File "/usr/local/lib/python3.9/site-packages/anyio/streams/memory.py", line 114, in receive
    raise EndOfStream
anyio.EndOfStream

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/chromadb/server/fastapi/__init__.py", line 80, in catch_exceptions_middleware
    return await call_next(request)
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/base.py", line 84, in call_next
    raise app_exc
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/base.py", line 70, in coro
    await self.app(scope, receive_or_disconnect, send_no_error)
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/base.py", line 108, in __call__
    response = await self.dispatch_func(request, call_next)
  File "/usr/local/lib/python3.9/site-packages/chromadb/server/fastapi/__init__.py", line 94, in check_http_version_middleware
    return await call_next(request)
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/base.py", line 84, in call_next
    raise app_exc
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/base.py", line 70, in coro
    await self.app(scope, receive_or_disconnect, send_no_error)
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/exceptions.py", line 79, in __call__
    raise exc
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/exceptions.py", line 68, in __call__
    await self.app(scope, receive, sender)
  File "/usr/local/lib/python3.9/site-packages/fastapi/middleware/asyncexitstack.py", line 20, in __call__
    raise e
  File "/usr/local/lib/python3.9/site-packages/fastapi/middleware/asyncexitstack.py", line 17, in __call__
    await self.app(scope, receive, send)
  File "/usr/local/lib/python3.9/site-packages/starlette/routing.py", line 718, in __call__
    await route.handle(scope, receive, send)
  File "/usr/local/lib/python3.9/site-packages/starlette/routing.py", line 276, in handle
    await self.app(scope, receive, send)
  File "/usr/local/lib/python3.9/site-packages/starlette/routing.py", line 66, in app
    response = await func(request)
  File "/usr/local/lib/python3.9/site-packages/fastapi/routing.py", line 273, in app
    raw_response = await run_endpoint_function(
  File "/usr/local/lib/python3.9/site-packages/fastapi/routing.py", line 192, in run_endpoint_function
    return await run_in_threadpool(dependant.call, **values)
  File "/usr/local/lib/python3.9/site-packages/starlette/concurrency.py", line 41, in run_in_threadpool
    return await anyio.to_thread.run_sync(func, *args)
  File "/usr/local/lib/python3.9/site-packages/anyio/to_thread.py", line 31, in run_sync
    return await get_asynclib().run_sync_in_worker_thread(
  File "/usr/local/lib/python3.9/site-packages/anyio/_backends/_asyncio.py", line 937, in run_sync_in_worker_thread
    return await future
  File "/usr/local/lib/python3.9/site-packages/anyio/_backends/_asyncio.py", line 867, in run
    result = context.run(func, *args)
  File "/usr/local/lib/python3.9/site-packages/chromadb/telemetry/opentelemetry/__init__.py", line 127, in wrapper
    return f(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/chromadb/auth/fastapi.py", line 234, in wrapped
    return f(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/chromadb/server/fastapi/__init__.py", line 433, in get_collection
    return self._api.get_collection(
  File "/usr/local/lib/python3.9/site-packages/chromadb/telemetry/opentelemetry/__init__.py", line 127, in wrapper
    return f(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/chromadb/api/segment.py", line 260, in get_collection
    raise ValueError(f"Collection {name} does not exist.")
ValueError: Collection movie_reviews does not exist.
[32mINFO[0m:     [29-12-2023 13:12:42] ::1:53167 - "GET /api/v1/collections/movie_reviews?tenant=default_tenant&database=default_database HTTP/1.1" 500
[31mERROR[0m:    [29-12-2023 13:12:47] Collection movie_reviews does not exist.
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/anyio/streams/memory.py", line 94, in receive
    return self.receive_nowait()
  File "/usr/local/lib/python3.9/site-packages/anyio/streams/memory.py", line 89, in receive_nowait
    raise WouldBlock
anyio.WouldBlock

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/base.py", line 78, in call_next
    message = await recv_stream.receive()
  File "/usr/local/lib/python3.9/site-packages/anyio/streams/memory.py", line 114, in receive
    raise EndOfStream
anyio.EndOfStream

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/chromadb/server/fastapi/__init__.py", line 80, in catch_exceptions_middleware
    return await call_next(request)
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/base.py", line 84, in call_next
    raise app_exc
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/base.py", line 70, in coro
    await self.app(scope, receive_or_disconnect, send_no_error)
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/base.py", line 108, in __call__
    response = await self.dispatch_func(request, call_next)
  File "/usr/local/lib/python3.9/site-packages/chromadb/server/fastapi/__init__.py", line 94, in check_http_version_middleware
    return await call_next(request)
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/base.py", line 84, in call_next
    raise app_exc
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/base.py", line 70, in coro
    await self.app(scope, receive_or_disconnect, send_no_error)
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/exceptions.py", line 79, in __call__
    raise exc
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/exceptions.py", line 68, in __call__
    await self.app(scope, receive, sender)
  File "/usr/local/lib/python3.9/site-packages/fastapi/middleware/asyncexitstack.py", line 20, in __call__
    raise e
  File "/usr/local/lib/python3.9/site-packages/fastapi/middleware/asyncexitstack.py", line 17, in __call__
    await self.app(scope, receive, send)
  File "/usr/local/lib/python3.9/site-packages/starlette/routing.py", line 718, in __call__
    await route.handle(scope, receive, send)
  File "/usr/local/lib/python3.9/site-packages/starlette/routing.py", line 276, in handle
    await self.app(scope, receive, send)
  File "/usr/local/lib/python3.9/site-packages/starlette/routing.py", line 66, in app
    response = await func(request)
  File "/usr/local/lib/python3.9/site-packages/fastapi/routing.py", line 273, in app
    raw_response = await run_endpoint_function(
  File "/usr/local/lib/python3.9/site-packages/fastapi/routing.py", line 192, in run_endpoint_function
    return await run_in_threadpool(dependant.call, **values)
  File "/usr/local/lib/python3.9/site-packages/starlette/concurrency.py", line 41, in run_in_threadpool
    return await anyio.to_thread.run_sync(func, *args)
  File "/usr/local/lib/python3.9/site-packages/anyio/to_thread.py", line 31, in run_sync
    return await get_asynclib().run_sync_in_worker_thread(
  File "/usr/local/lib/python3.9/site-packages/anyio/_backends/_asyncio.py", line 937, in run_sync_in_worker_thread
    return await future
  File "/usr/local/lib/python3.9/site-packages/anyio/_backends/_asyncio.py", line 867, in run
    result = context.run(func, *args)
  File "/usr/local/lib/python3.9/site-packages/chromadb/telemetry/opentelemetry/__init__.py", line 127, in wrapper
    return f(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/chromadb/auth/fastapi.py", line 234, in wrapped
    return f(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/chromadb/server/fastapi/__init__.py", line 433, in get_collection
    return self._api.get_collection(
  File "/usr/local/lib/python3.9/site-packages/chromadb/telemetry/opentelemetry/__init__.py", line 127, in wrapper
    return f(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/chromadb/api/segment.py", line 260, in get_collection
    raise ValueError(f"Collection {name} does not exist.")
ValueError: Collection movie_reviews does not exist.
[32mINFO[0m:     [29-12-2023 13:12:47] ::1:53168 - "GET /api/v1/collections/movie_reviews?tenant=default_tenant&database=default_database HTTP/1.1" 500
[32mINFO[0m:     [29-12-2023 13:13:20] Shutting down
[32mINFO[0m:     [29-12-2023 13:13:20] Waiting for application shutdown.
[32mINFO[0m:     [29-12-2023 13:13:20] Application shutdown complete.
[32mINFO[0m:     [29-12-2023 13:13:20] Finished server process [[36m49875[0m]
[32mINFO[0m:     [29-12-2023 13:13:21] Set chroma_server_nofile to 65535
[32mINFO[0m:     [29-12-2023 13:13:21] Anonymized telemetry enabled. See                     https://docs.trychroma.com/telemetry for more information.
[36mDEBUG[0m:    [29-12-2023 13:13:21] Starting component System
[36mDEBUG[0m:    [29-12-2023 13:13:21] Starting component OpenTelemetryClient
[36mDEBUG[0m:    [29-12-2023 13:13:21] Starting component SimpleAssignmentPolicy
[36mDEBUG[0m:    [29-12-2023 13:13:21] Starting component SqliteDB
[36mDEBUG[0m:    [29-12-2023 13:13:21] Starting component Posthog
[36mDEBUG[0m:    [29-12-2023 13:13:21] Starting component LocalSegmentManager
[36mDEBUG[0m:    [29-12-2023 13:13:21] Starting component SegmentAPI
[32mINFO[0m:     [29-12-2023 13:13:22] Started server process [[36m50012[0m]
[32mINFO[0m:     [29-12-2023 13:13:22] Waiting for application startup.
[32mINFO[0m:     [29-12-2023 13:13:22] Application startup complete.
[32mINFO[0m:     [29-12-2023 13:13:22] Uvicorn running on [1mhttp://localhost:8000[0m (Press CTRL+C to quit)
[31mERROR[0m:    [29-12-2023 13:13:24] Collection movie_reviews does not exist.
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/anyio/streams/memory.py", line 94, in receive
    return self.receive_nowait()
  File "/usr/local/lib/python3.9/site-packages/anyio/streams/memory.py", line 89, in receive_nowait
    raise WouldBlock
anyio.WouldBlock

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/base.py", line 78, in call_next
    message = await recv_stream.receive()
  File "/usr/local/lib/python3.9/site-packages/anyio/streams/memory.py", line 114, in receive
    raise EndOfStream
anyio.EndOfStream

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/chromadb/server/fastapi/__init__.py", line 80, in catch_exceptions_middleware
    return await call_next(request)
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/base.py", line 84, in call_next
    raise app_exc
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/base.py", line 70, in coro
    await self.app(scope, receive_or_disconnect, send_no_error)
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/base.py", line 108, in __call__
    response = await self.dispatch_func(request, call_next)
  File "/usr/local/lib/python3.9/site-packages/chromadb/server/fastapi/__init__.py", line 94, in check_http_version_middleware
    return await call_next(request)
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/base.py", line 84, in call_next
    raise app_exc
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/base.py", line 70, in coro
    await self.app(scope, receive_or_disconnect, send_no_error)
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/exceptions.py", line 79, in __call__
    raise exc
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/exceptions.py", line 68, in __call__
    await self.app(scope, receive, sender)
  File "/usr/local/lib/python3.9/site-packages/fastapi/middleware/asyncexitstack.py", line 20, in __call__
    raise e
  File "/usr/local/lib/python3.9/site-packages/fastapi/middleware/asyncexitstack.py", line 17, in __call__
    await self.app(scope, receive, send)
  File "/usr/local/lib/python3.9/site-packages/starlette/routing.py", line 718, in __call__
    await route.handle(scope, receive, send)
  File "/usr/local/lib/python3.9/site-packages/starlette/routing.py", line 276, in handle
    await self.app(scope, receive, send)
  File "/usr/local/lib/python3.9/site-packages/starlette/routing.py", line 66, in app
    response = await func(request)
  File "/usr/local/lib/python3.9/site-packages/fastapi/routing.py", line 273, in app
    raw_response = await run_endpoint_function(
  File "/usr/local/lib/python3.9/site-packages/fastapi/routing.py", line 192, in run_endpoint_function
    return await run_in_threadpool(dependant.call, **values)
  File "/usr/local/lib/python3.9/site-packages/starlette/concurrency.py", line 41, in run_in_threadpool
    return await anyio.to_thread.run_sync(func, *args)
  File "/usr/local/lib/python3.9/site-packages/anyio/to_thread.py", line 31, in run_sync
    return await get_asynclib().run_sync_in_worker_thread(
  File "/usr/local/lib/python3.9/site-packages/anyio/_backends/_asyncio.py", line 937, in run_sync_in_worker_thread
    return await future
  File "/usr/local/lib/python3.9/site-packages/anyio/_backends/_asyncio.py", line 867, in run
    result = context.run(func, *args)
  File "/usr/local/lib/python3.9/site-packages/chromadb/telemetry/opentelemetry/__init__.py", line 127, in wrapper
    return f(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/chromadb/auth/fastapi.py", line 234, in wrapped
    return f(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/chromadb/server/fastapi/__init__.py", line 433, in get_collection
    return self._api.get_collection(
  File "/usr/local/lib/python3.9/site-packages/chromadb/telemetry/opentelemetry/__init__.py", line 127, in wrapper
    return f(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/chromadb/api/segment.py", line 260, in get_collection
    raise ValueError(f"Collection {name} does not exist.")
ValueError: Collection movie_reviews does not exist.
[32mINFO[0m:     [29-12-2023 13:13:24] ::1:53173 - "GET /api/v1/collections/movie_reviews?tenant=default_tenant&database=default_database HTTP/1.1" 500
[32mINFO[0m:     [29-12-2023 13:13:42] Shutting down
[32mINFO[0m:     [29-12-2023 13:13:42] Waiting for application shutdown.
[32mINFO[0m:     [29-12-2023 13:13:42] Application shutdown complete.
[32mINFO[0m:     [29-12-2023 13:13:42] Finished server process [[36m50012[0m]
[32mINFO[0m:     [29-12-2023 13:13:49] Set chroma_server_nofile to 65535
[32mINFO[0m:     [29-12-2023 13:13:49] Anonymized telemetry enabled. See                     https://docs.trychroma.com/telemetry for more information.
[36mDEBUG[0m:    [29-12-2023 13:13:49] Starting component System
[36mDEBUG[0m:    [29-12-2023 13:13:49] Starting component OpenTelemetryClient
[36mDEBUG[0m:    [29-12-2023 13:13:49] Starting component SimpleAssignmentPolicy
[36mDEBUG[0m:    [29-12-2023 13:13:49] Starting component SqliteDB
[36mDEBUG[0m:    [29-12-2023 13:13:49] Starting component Posthog
[36mDEBUG[0m:    [29-12-2023 13:13:49] Starting component LocalSegmentManager
[36mDEBUG[0m:    [29-12-2023 13:13:49] Starting component SegmentAPI
[32mINFO[0m:     [29-12-2023 13:13:49] Started server process [[36m50187[0m]
[32mINFO[0m:     [29-12-2023 13:13:49] Waiting for application startup.
[32mINFO[0m:     [29-12-2023 13:13:49] Application startup complete.
[32mINFO[0m:     [29-12-2023 13:13:49] Uvicorn running on [1mhttp://localhost:8000[0m (Press CTRL+C to quit)
[32mINFO[0m:     [29-12-2023 13:13:52] ::1:53175 - "POST /api/v1/collections?tenant=default_tenant&database=default_database HTTP/1.1" 200
[32mINFO[0m:     [29-12-2023 13:13:52] ::1:53176 - "GET /api/v1/collections?tenant=default_tenant&database=default_database HTTP/1.1" 200
[32mINFO[0m:     [29-12-2023 13:13:52] ::1:53175 - "GET /api/v1/collections/movie_reviews?tenant=default_tenant&database=default_database HTTP/1.1" 200
[31mERROR[0m:    [29-12-2023 13:18:16] Collection movie_reviews already exists
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/anyio/streams/memory.py", line 94, in receive
    return self.receive_nowait()
  File "/usr/local/lib/python3.9/site-packages/anyio/streams/memory.py", line 89, in receive_nowait
    raise WouldBlock
anyio.WouldBlock

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/base.py", line 78, in call_next
    message = await recv_stream.receive()
  File "/usr/local/lib/python3.9/site-packages/anyio/streams/memory.py", line 114, in receive
    raise EndOfStream
anyio.EndOfStream

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/chromadb/server/fastapi/__init__.py", line 80, in catch_exceptions_middleware
    return await call_next(request)
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/base.py", line 84, in call_next
    raise app_exc
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/base.py", line 70, in coro
    await self.app(scope, receive_or_disconnect, send_no_error)
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/base.py", line 108, in __call__
    response = await self.dispatch_func(request, call_next)
  File "/usr/local/lib/python3.9/site-packages/chromadb/server/fastapi/__init__.py", line 94, in check_http_version_middleware
    return await call_next(request)
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/base.py", line 84, in call_next
    raise app_exc
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/base.py", line 70, in coro
    await self.app(scope, receive_or_disconnect, send_no_error)
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/exceptions.py", line 79, in __call__
    raise exc
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/exceptions.py", line 68, in __call__
    await self.app(scope, receive, sender)
  File "/usr/local/lib/python3.9/site-packages/fastapi/middleware/asyncexitstack.py", line 20, in __call__
    raise e
  File "/usr/local/lib/python3.9/site-packages/fastapi/middleware/asyncexitstack.py", line 17, in __call__
    await self.app(scope, receive, send)
  File "/usr/local/lib/python3.9/site-packages/starlette/routing.py", line 718, in __call__
    await route.handle(scope, receive, send)
  File "/usr/local/lib/python3.9/site-packages/starlette/routing.py", line 276, in handle
    await self.app(scope, receive, send)
  File "/usr/local/lib/python3.9/site-packages/starlette/routing.py", line 66, in app
    response = await func(request)
  File "/usr/local/lib/python3.9/site-packages/fastapi/routing.py", line 273, in app
    raw_response = await run_endpoint_function(
  File "/usr/local/lib/python3.9/site-packages/fastapi/routing.py", line 192, in run_endpoint_function
    return await run_in_threadpool(dependant.call, **values)
  File "/usr/local/lib/python3.9/site-packages/starlette/concurrency.py", line 41, in run_in_threadpool
    return await anyio.to_thread.run_sync(func, *args)
  File "/usr/local/lib/python3.9/site-packages/anyio/to_thread.py", line 31, in run_sync
    return await get_asynclib().run_sync_in_worker_thread(
  File "/usr/local/lib/python3.9/site-packages/anyio/_backends/_asyncio.py", line 937, in run_sync_in_worker_thread
    return await future
  File "/usr/local/lib/python3.9/site-packages/anyio/_backends/_asyncio.py", line 867, in run
    result = context.run(func, *args)
  File "/usr/local/lib/python3.9/site-packages/chromadb/telemetry/opentelemetry/__init__.py", line 127, in wrapper
    return f(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/chromadb/auth/fastapi.py", line 234, in wrapped
    return f(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/chromadb/server/fastapi/__init__.py", line 408, in create_collection
    return self._api.create_collection(
  File "/usr/local/lib/python3.9/site-packages/chromadb/telemetry/opentelemetry/__init__.py", line 127, in wrapper
    return f(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/chromadb/api/segment.py", line 166, in create_collection
    coll, created = self._sysdb.create_collection(
  File "/usr/local/lib/python3.9/site-packages/chromadb/telemetry/opentelemetry/__init__.py", line 127, in wrapper
    return f(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/chromadb/db/mixins/sysdb.py", line 225, in create_collection
    raise UniqueConstraintError(f"Collection {name} already exists")
chromadb.db.base.UniqueConstraintError: Collection movie_reviews already exists
[32mINFO[0m:     [29-12-2023 13:18:16] ::1:53212 - "POST /api/v1/collections?tenant=default_tenant&database=default_database HTTP/1.1" 500
[32mINFO[0m:     [29-12-2023 13:18:25] Shutting down
[32mINFO[0m:     [29-12-2023 13:18:25] Waiting for application shutdown.
[32mINFO[0m:     [29-12-2023 13:18:25] Application shutdown complete.
[32mINFO[0m:     [29-12-2023 13:18:25] Finished server process [[36m50187[0m]
[32mINFO[0m:     [29-12-2023 13:18:27] Set chroma_server_nofile to 65535
[32mINFO[0m:     [29-12-2023 13:18:27] Anonymized telemetry enabled. See                     https://docs.trychroma.com/telemetry for more information.
[36mDEBUG[0m:    [29-12-2023 13:18:27] Starting component System
[36mDEBUG[0m:    [29-12-2023 13:18:27] Starting component OpenTelemetryClient
[36mDEBUG[0m:    [29-12-2023 13:18:27] Starting component SimpleAssignmentPolicy
[36mDEBUG[0m:    [29-12-2023 13:18:27] Starting component SqliteDB
[36mDEBUG[0m:    [29-12-2023 13:18:27] Starting component Posthog
[36mDEBUG[0m:    [29-12-2023 13:18:27] Starting component LocalSegmentManager
[36mDEBUG[0m:    [29-12-2023 13:18:27] Starting component SegmentAPI
[32mINFO[0m:     [29-12-2023 13:18:27] Started server process [[36m50951[0m]
[32mINFO[0m:     [29-12-2023 13:18:27] Waiting for application startup.
[32mINFO[0m:     [29-12-2023 13:18:27] Application startup complete.
[32mINFO[0m:     [29-12-2023 13:18:27] Uvicorn running on [1mhttp://localhost:8000[0m (Press CTRL+C to quit)
[32mINFO[0m:     [29-12-2023 13:18:30] ::1:53214 - "POST /api/v1/collections?tenant=default_tenant&database=default_database HTTP/1.1" 200
[32mINFO[0m:     [29-12-2023 13:18:30] ::1:53215 - "GET /api/v1/collections?tenant=default_tenant&database=default_database HTTP/1.1" 200
[32mINFO[0m:     [29-12-2023 13:18:30] ::1:53214 - "GET /api/v1/collections/movie_reviews?tenant=default_tenant&database=default_database HTTP/1.1" 200
[36mDEBUG[0m:    [29-12-2023 13:18:49] Starting component PersistentLocalHnswSegment
[32mINFO[0m:     [29-12-2023 13:18:49] ::1:53218 - "POST /api/v1/collections/84ad54f9-487c-454a-aa3b-52ecd97c66a7/add HTTP/1.1" 201
[32mINFO[0m:     [29-12-2023 13:31:26] Shutting down
[32mINFO[0m:     [29-12-2023 13:31:26] Waiting for application shutdown.
[32mINFO[0m:     [29-12-2023 13:31:26] Application shutdown complete.
[32mINFO[0m:     [29-12-2023 13:31:26] Finished server process [[36m50951[0m]
[32mINFO[0m:     [29-12-2023 13:31:35] Set chroma_server_nofile to 65535
[32mINFO[0m:     [29-12-2023 13:31:35] Anonymized telemetry enabled. See                     https://docs.trychroma.com/telemetry for more information.
[36mDEBUG[0m:    [29-12-2023 13:31:35] Starting component System
[36mDEBUG[0m:    [29-12-2023 13:31:35] Starting component OpenTelemetryClient
[36mDEBUG[0m:    [29-12-2023 13:31:35] Starting component SimpleAssignmentPolicy
[36mDEBUG[0m:    [29-12-2023 13:31:35] Starting component SqliteDB
[36mDEBUG[0m:    [29-12-2023 13:31:35] Starting component Posthog
[36mDEBUG[0m:    [29-12-2023 13:31:35] Starting component LocalSegmentManager
[36mDEBUG[0m:    [29-12-2023 13:31:35] Starting component SegmentAPI
[32mINFO[0m:     [29-12-2023 13:31:35] Started server process [[36m53283[0m]
[32mINFO[0m:     [29-12-2023 13:31:35] Waiting for application startup.
[32mINFO[0m:     [29-12-2023 13:31:35] Application startup complete.
[32mINFO[0m:     [29-12-2023 13:31:35] Uvicorn running on [1mhttp://localhost:8000[0m (Press CTRL+C to quit)
[32mINFO[0m:     [29-12-2023 13:31:38] ::1:53272 - "POST /api/v1/collections?tenant=default_tenant&database=default_database HTTP/1.1" 200
[32mINFO[0m:     [29-12-2023 13:31:38] ::1:53273 - "GET /api/v1/collections?tenant=default_tenant&database=default_database HTTP/1.1" 200
[32mINFO[0m:     [29-12-2023 13:31:38] ::1:53272 - "GET /api/v1/collections/movie_reviews?tenant=default_tenant&database=default_database HTTP/1.1" 200
[36mDEBUG[0m:    [29-12-2023 13:31:56] Starting component PersistentLocalHnswSegment
[32mINFO[0m:     [29-12-2023 13:31:56] ::1:53276 - "POST /api/v1/collections/59ba19d8-b2b0-4dac-b014-34bfc3aeb713/add HTTP/1.1" 201
[33mWARNING[0m:  [29-12-2023 13:31:56] Delete of nonexisting embedding ID: 5768bf05-f70e-4f6a-895d-a682ff9a99a9
[33mWARNING[0m:  [29-12-2023 13:31:56] Delete of nonexisting embedding ID: 5768bf05-f70e-4f6a-895d-a682ff9a99a9
[32mINFO[0m:     [29-12-2023 13:31:56] ::1:53277 - "POST /api/v1/collections/59ba19d8-b2b0-4dac-b014-34bfc3aeb713/delete HTTP/1.1" 200
[31mERROR[0m:    [29-12-2023 15:07:36] Collection movie_reviews already exists
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/anyio/streams/memory.py", line 94, in receive
    return self.receive_nowait()
  File "/usr/local/lib/python3.9/site-packages/anyio/streams/memory.py", line 89, in receive_nowait
    raise WouldBlock
anyio.WouldBlock

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/base.py", line 78, in call_next
    message = await recv_stream.receive()
  File "/usr/local/lib/python3.9/site-packages/anyio/streams/memory.py", line 114, in receive
    raise EndOfStream
anyio.EndOfStream

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/chromadb/server/fastapi/__init__.py", line 80, in catch_exceptions_middleware
    return await call_next(request)
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/base.py", line 84, in call_next
    raise app_exc
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/base.py", line 70, in coro
    await self.app(scope, receive_or_disconnect, send_no_error)
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/base.py", line 108, in __call__
    response = await self.dispatch_func(request, call_next)
  File "/usr/local/lib/python3.9/site-packages/chromadb/server/fastapi/__init__.py", line 94, in check_http_version_middleware
    return await call_next(request)
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/base.py", line 84, in call_next
    raise app_exc
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/base.py", line 70, in coro
    await self.app(scope, receive_or_disconnect, send_no_error)
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/exceptions.py", line 79, in __call__
    raise exc
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/exceptions.py", line 68, in __call__
    await self.app(scope, receive, sender)
  File "/usr/local/lib/python3.9/site-packages/fastapi/middleware/asyncexitstack.py", line 20, in __call__
    raise e
  File "/usr/local/lib/python3.9/site-packages/fastapi/middleware/asyncexitstack.py", line 17, in __call__
    await self.app(scope, receive, send)
  File "/usr/local/lib/python3.9/site-packages/starlette/routing.py", line 718, in __call__
    await route.handle(scope, receive, send)
  File "/usr/local/lib/python3.9/site-packages/starlette/routing.py", line 276, in handle
    await self.app(scope, receive, send)
  File "/usr/local/lib/python3.9/site-packages/starlette/routing.py", line 66, in app
    response = await func(request)
  File "/usr/local/lib/python3.9/site-packages/fastapi/routing.py", line 273, in app
    raw_response = await run_endpoint_function(
  File "/usr/local/lib/python3.9/site-packages/fastapi/routing.py", line 192, in run_endpoint_function
    return await run_in_threadpool(dependant.call, **values)
  File "/usr/local/lib/python3.9/site-packages/starlette/concurrency.py", line 41, in run_in_threadpool
    return await anyio.to_thread.run_sync(func, *args)
  File "/usr/local/lib/python3.9/site-packages/anyio/to_thread.py", line 31, in run_sync
    return await get_asynclib().run_sync_in_worker_thread(
  File "/usr/local/lib/python3.9/site-packages/anyio/_backends/_asyncio.py", line 937, in run_sync_in_worker_thread
    return await future
  File "/usr/local/lib/python3.9/site-packages/anyio/_backends/_asyncio.py", line 867, in run
    result = context.run(func, *args)
  File "/usr/local/lib/python3.9/site-packages/chromadb/telemetry/opentelemetry/__init__.py", line 127, in wrapper
    return f(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/chromadb/auth/fastapi.py", line 234, in wrapped
    return f(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/chromadb/server/fastapi/__init__.py", line 408, in create_collection
    return self._api.create_collection(
  File "/usr/local/lib/python3.9/site-packages/chromadb/telemetry/opentelemetry/__init__.py", line 127, in wrapper
    return f(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/chromadb/api/segment.py", line 166, in create_collection
    coll, created = self._sysdb.create_collection(
  File "/usr/local/lib/python3.9/site-packages/chromadb/telemetry/opentelemetry/__init__.py", line 127, in wrapper
    return f(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/chromadb/db/mixins/sysdb.py", line 225, in create_collection
    raise UniqueConstraintError(f"Collection {name} already exists")
chromadb.db.base.UniqueConstraintError: Collection movie_reviews already exists
[32mINFO[0m:     [29-12-2023 15:07:36] ::1:53734 - "POST /api/v1/collections?tenant=default_tenant&database=default_database HTTP/1.1" 500
[32mINFO[0m:     [29-12-2023 18:26:49] Shutting down
[32mINFO[0m:     [29-12-2023 18:26:49] Waiting for application shutdown.
[32mINFO[0m:     [29-12-2023 18:26:49] Application shutdown complete.
[32mINFO[0m:     [29-12-2023 18:26:49] Finished server process [[36m53283[0m]
[32mINFO[0m:     [29-12-2023 18:26:56] Set chroma_server_nofile to 65535
[32mINFO[0m:     [29-12-2023 18:26:56] Anonymized telemetry enabled. See                     https://docs.trychroma.com/telemetry for more information.
[36mDEBUG[0m:    [29-12-2023 18:26:56] Starting component System
[36mDEBUG[0m:    [29-12-2023 18:26:56] Starting component OpenTelemetryClient
[36mDEBUG[0m:    [29-12-2023 18:26:56] Starting component SimpleAssignmentPolicy
[36mDEBUG[0m:    [29-12-2023 18:26:56] Starting component SqliteDB
[36mDEBUG[0m:    [29-12-2023 18:26:56] Starting component Posthog
[36mDEBUG[0m:    [29-12-2023 18:26:56] Starting component LocalSegmentManager
[36mDEBUG[0m:    [29-12-2023 18:26:56] Starting component SegmentAPI
[32mINFO[0m:     [29-12-2023 18:26:56] Started server process [[36m98044[0m]
[32mINFO[0m:     [29-12-2023 18:26:56] Waiting for application startup.
[32mINFO[0m:     [29-12-2023 18:26:56] Application startup complete.
[32mINFO[0m:     [29-12-2023 18:26:56] Uvicorn running on [1mhttp://localhost:8000[0m (Press CTRL+C to quit)
[32mINFO[0m:     [29-12-2023 18:27:00] ::1:54635 - "POST /api/v1/collections?tenant=default_tenant&database=default_database HTTP/1.1" 200
[32mINFO[0m:     [29-12-2023 18:27:01] ::1:54636 - "GET /api/v1/collections?tenant=default_tenant&database=default_database HTTP/1.1" 200
[32mINFO[0m:     [29-12-2023 18:27:01] ::1:54635 - "GET /api/v1/collections/movie_reviews?tenant=default_tenant&database=default_database HTTP/1.1" 200
[36mDEBUG[0m:    [29-12-2023 18:27:20] Starting component PersistentLocalHnswSegment
[32mINFO[0m:     [29-12-2023 18:27:20] ::1:54641 - "POST /api/v1/collections/6231de9e-a811-4fdf-bb58-fcd9f4460a0c/add HTTP/1.1" 201
[33mWARNING[0m:  [29-12-2023 18:27:20] Delete of nonexisting embedding ID: 5768bf05-f70e-4f6a-895d-a682ff9a99a9
[33mWARNING[0m:  [29-12-2023 18:27:20] Delete of nonexisting embedding ID: 5768bf05-f70e-4f6a-895d-a682ff9a99a9
[32mINFO[0m:     [29-12-2023 18:27:20] ::1:54642 - "POST /api/v1/collections/6231de9e-a811-4fdf-bb58-fcd9f4460a0c/delete HTTP/1.1" 200
[32mINFO[0m:     [29-12-2023 18:48:41] Shutting down
[32mINFO[0m:     [29-12-2023 18:48:41] Waiting for application shutdown.
[32mINFO[0m:     [29-12-2023 18:48:41] Application shutdown complete.
[32mINFO[0m:     [29-12-2023 18:48:41] Finished server process [[36m98044[0m]
[32mINFO[0m:     [29-12-2023 18:48:49] Set chroma_server_nofile to 65535
[32mINFO[0m:     [29-12-2023 18:48:49] Anonymized telemetry enabled. See                     https://docs.trychroma.com/telemetry for more information.
[36mDEBUG[0m:    [29-12-2023 18:48:49] Starting component System
[36mDEBUG[0m:    [29-12-2023 18:48:49] Starting component OpenTelemetryClient
[36mDEBUG[0m:    [29-12-2023 18:48:49] Starting component SimpleAssignmentPolicy
[36mDEBUG[0m:    [29-12-2023 18:48:49] Starting component SqliteDB
[36mDEBUG[0m:    [29-12-2023 18:48:49] Starting component Posthog
[36mDEBUG[0m:    [29-12-2023 18:48:49] Starting component LocalSegmentManager
[36mDEBUG[0m:    [29-12-2023 18:48:49] Starting component SegmentAPI
[32mINFO[0m:     [29-12-2023 18:48:49] Started server process [[36m2102[0m]
[32mINFO[0m:     [29-12-2023 18:48:49] Waiting for application startup.
[32mINFO[0m:     [29-12-2023 18:48:49] Application startup complete.
[32mINFO[0m:     [29-12-2023 18:48:49] Uvicorn running on [1mhttp://localhost:8000[0m (Press CTRL+C to quit)
[32mINFO[0m:     [29-12-2023 18:48:53] ::1:54708 - "POST /api/v1/collections?tenant=default_tenant&database=default_database HTTP/1.1" 200
[32mINFO[0m:     [29-12-2023 18:48:53] ::1:54709 - "GET /api/v1/collections?tenant=default_tenant&database=default_database HTTP/1.1" 200
[32mINFO[0m:     [29-12-2023 18:48:53] ::1:54708 - "GET /api/v1/collections/movie_reviews?tenant=default_tenant&database=default_database HTTP/1.1" 200
[36mDEBUG[0m:    [29-12-2023 18:49:12] Starting component PersistentLocalHnswSegment
[32mINFO[0m:     [29-12-2023 18:49:13] ::1:54714 - "POST /api/v1/collections/7f092c30-1b83-4073-9f06-59e817c30412/add HTTP/1.1" 201
[33mWARNING[0m:  [29-12-2023 18:49:13] Delete of nonexisting embedding ID: 5768bf05-f70e-4f6a-895d-a682ff9a99a9
[33mWARNING[0m:  [29-12-2023 18:49:13] Delete of nonexisting embedding ID: 5768bf05-f70e-4f6a-895d-a682ff9a99a9
[32mINFO[0m:     [29-12-2023 18:49:13] ::1:54715 - "POST /api/v1/collections/7f092c30-1b83-4073-9f06-59e817c30412/delete HTTP/1.1" 200
[32mINFO[0m:     [29-12-2023 18:49:13] ::1:54714 - "POST /api/v1/collections/7f092c30-1b83-4073-9f06-59e817c30412/query HTTP/1.1" 400
[32mINFO[0m:     [29-12-2023 18:51:32] Shutting down
[32mINFO[0m:     [29-12-2023 18:51:32] Waiting for application shutdown.
[32mINFO[0m:     [29-12-2023 18:51:32] Application shutdown complete.
[32mINFO[0m:     [29-12-2023 18:51:32] Finished server process [[36m2102[0m]
[32mINFO[0m:     [29-12-2023 18:51:38] Set chroma_server_nofile to 65535
[32mINFO[0m:     [29-12-2023 18:51:38] Anonymized telemetry enabled. See                     https://docs.trychroma.com/telemetry for more information.
[36mDEBUG[0m:    [29-12-2023 18:51:38] Starting component System
[36mDEBUG[0m:    [29-12-2023 18:51:38] Starting component OpenTelemetryClient
[36mDEBUG[0m:    [29-12-2023 18:51:38] Starting component SimpleAssignmentPolicy
[36mDEBUG[0m:    [29-12-2023 18:51:38] Starting component SqliteDB
[36mDEBUG[0m:    [29-12-2023 18:51:38] Starting component Posthog
[36mDEBUG[0m:    [29-12-2023 18:51:38] Starting component LocalSegmentManager
[36mDEBUG[0m:    [29-12-2023 18:51:38] Starting component SegmentAPI
[32mINFO[0m:     [29-12-2023 18:51:38] Started server process [[36m2668[0m]
[32mINFO[0m:     [29-12-2023 18:51:38] Waiting for application startup.
[32mINFO[0m:     [29-12-2023 18:51:38] Application startup complete.
[32mINFO[0m:     [29-12-2023 18:51:38] Uvicorn running on [1mhttp://localhost:8000[0m (Press CTRL+C to quit)
[32mINFO[0m:     [29-12-2023 18:51:46] ::1:54735 - "POST /api/v1/collections?tenant=default_tenant&database=default_database HTTP/1.1" 200
[32mINFO[0m:     [29-12-2023 18:51:46] ::1:54736 - "GET /api/v1/collections?tenant=default_tenant&database=default_database HTTP/1.1" 200
[32mINFO[0m:     [29-12-2023 18:51:46] ::1:54735 - "GET /api/v1/collections/movie_reviews?tenant=default_tenant&database=default_database HTTP/1.1" 200
[36mDEBUG[0m:    [29-12-2023 18:52:05] Starting component PersistentLocalHnswSegment
[32mINFO[0m:     [29-12-2023 18:52:05] ::1:54739 - "POST /api/v1/collections/ad200496-5b25-42e3-a886-db82a80de6b8/add HTTP/1.1" 201
[33mWARNING[0m:  [29-12-2023 18:52:05] Delete of nonexisting embedding ID: 5768bf05-f70e-4f6a-895d-a682ff9a99a9
[33mWARNING[0m:  [29-12-2023 18:52:05] Delete of nonexisting embedding ID: 5768bf05-f70e-4f6a-895d-a682ff9a99a9
[32mINFO[0m:     [29-12-2023 18:52:05] ::1:54740 - "POST /api/v1/collections/ad200496-5b25-42e3-a886-db82a80de6b8/delete HTTP/1.1" 200
[32mINFO[0m:     [29-12-2023 18:53:34] Shutting down
[32mINFO[0m:     [29-12-2023 18:53:34] Waiting for application shutdown.
[32mINFO[0m:     [29-12-2023 18:53:35] Application shutdown complete.
[32mINFO[0m:     [29-12-2023 18:53:35] Finished server process [[36m2668[0m]
[32mINFO[0m:     [29-12-2023 18:53:37] Set chroma_server_nofile to 65535
[32mINFO[0m:     [29-12-2023 18:53:37] Anonymized telemetry enabled. See                     https://docs.trychroma.com/telemetry for more information.
[36mDEBUG[0m:    [29-12-2023 18:53:37] Starting component System
[36mDEBUG[0m:    [29-12-2023 18:53:37] Starting component OpenTelemetryClient
[36mDEBUG[0m:    [29-12-2023 18:53:37] Starting component SimpleAssignmentPolicy
[36mDEBUG[0m:    [29-12-2023 18:53:37] Starting component SqliteDB
[36mDEBUG[0m:    [29-12-2023 18:53:37] Starting component Posthog
[36mDEBUG[0m:    [29-12-2023 18:53:37] Starting component LocalSegmentManager
[36mDEBUG[0m:    [29-12-2023 18:53:37] Starting component SegmentAPI
[32mINFO[0m:     [29-12-2023 18:53:37] Started server process [[36m3005[0m]
[32mINFO[0m:     [29-12-2023 18:53:37] Waiting for application startup.
[32mINFO[0m:     [29-12-2023 18:53:37] Application startup complete.
[32mINFO[0m:     [29-12-2023 18:53:37] Uvicorn running on [1mhttp://localhost:8000[0m (Press CTRL+C to quit)
[32mINFO[0m:     [29-12-2023 18:53:44] ::1:54745 - "POST /api/v1/collections?tenant=default_tenant&database=default_database HTTP/1.1" 200
[32mINFO[0m:     [29-12-2023 18:53:44] ::1:54746 - "GET /api/v1/collections?tenant=default_tenant&database=default_database HTTP/1.1" 200
[32mINFO[0m:     [29-12-2023 18:53:44] ::1:54745 - "GET /api/v1/collections/movie_reviews?tenant=default_tenant&database=default_database HTTP/1.1" 200
[36mDEBUG[0m:    [29-12-2023 18:54:01] Starting component PersistentLocalHnswSegment
[32mINFO[0m:     [29-12-2023 18:54:01] ::1:54750 - "POST /api/v1/collections/16a535c5-f852-4113-b020-0eb29cefb7f5/add HTTP/1.1" 201
[33mWARNING[0m:  [29-12-2023 18:54:01] Delete of nonexisting embedding ID: 5768bf05-f70e-4f6a-895d-a682ff9a99a9
[33mWARNING[0m:  [29-12-2023 18:54:01] Delete of nonexisting embedding ID: 5768bf05-f70e-4f6a-895d-a682ff9a99a9
[32mINFO[0m:     [29-12-2023 18:54:01] ::1:54751 - "POST /api/v1/collections/16a535c5-f852-4113-b020-0eb29cefb7f5/delete HTTP/1.1" 200
[32mINFO[0m:     [29-12-2023 18:54:35] Shutting down
[32mINFO[0m:     [29-12-2023 18:54:35] Waiting for application shutdown.
[32mINFO[0m:     [29-12-2023 18:54:35] Application shutdown complete.
[32mINFO[0m:     [29-12-2023 18:54:35] Finished server process [[36m3005[0m]
[32mINFO[0m:     [29-12-2023 18:54:45] Set chroma_server_nofile to 65535
[32mINFO[0m:     [29-12-2023 18:54:45] Anonymized telemetry enabled. See                     https://docs.trychroma.com/telemetry for more information.
[36mDEBUG[0m:    [29-12-2023 18:54:45] Starting component System
[36mDEBUG[0m:    [29-12-2023 18:54:45] Starting component OpenTelemetryClient
[36mDEBUG[0m:    [29-12-2023 18:54:45] Starting component SimpleAssignmentPolicy
[36mDEBUG[0m:    [29-12-2023 18:54:45] Starting component SqliteDB
[36mDEBUG[0m:    [29-12-2023 18:54:45] Starting component Posthog
[36mDEBUG[0m:    [29-12-2023 18:54:45] Starting component LocalSegmentManager
[36mDEBUG[0m:    [29-12-2023 18:54:45] Starting component SegmentAPI
[32mINFO[0m:     [29-12-2023 18:54:45] Started server process [[36m3208[0m]
[32mINFO[0m:     [29-12-2023 18:54:45] Waiting for application startup.
[32mINFO[0m:     [29-12-2023 18:54:45] Application startup complete.
[32mINFO[0m:     [29-12-2023 18:54:45] Uvicorn running on [1mhttp://localhost:8000[0m (Press CTRL+C to quit)
[32mINFO[0m:     [29-12-2023 18:54:53] ::1:54753 - "POST /api/v1/collections?tenant=default_tenant&database=default_database HTTP/1.1" 200
[32mINFO[0m:     [29-12-2023 18:54:53] ::1:54754 - "GET /api/v1/collections?tenant=default_tenant&database=default_database HTTP/1.1" 200
[32mINFO[0m:     [29-12-2023 18:54:53] ::1:54753 - "GET /api/v1/collections/movie_reviews?tenant=default_tenant&database=default_database HTTP/1.1" 200
[36mDEBUG[0m:    [29-12-2023 18:55:11] Starting component PersistentLocalHnswSegment
[32mINFO[0m:     [29-12-2023 18:55:12] ::1:54759 - "POST /api/v1/collections/e1d5c0c6-56c0-441e-b3e5-c48e1c51f1f1/add HTTP/1.1" 201
[33mWARNING[0m:  [29-12-2023 18:55:12] Delete of nonexisting embedding ID: 5768bf05-f70e-4f6a-895d-a682ff9a99a9
[33mWARNING[0m:  [29-12-2023 18:55:12] Delete of nonexisting embedding ID: 5768bf05-f70e-4f6a-895d-a682ff9a99a9
[32mINFO[0m:     [29-12-2023 18:55:12] ::1:54760 - "POST /api/v1/collections/e1d5c0c6-56c0-441e-b3e5-c48e1c51f1f1/delete HTTP/1.1" 200
[32mINFO[0m:     [29-12-2023 18:55:12] ::1:54759 - "POST /api/v1/collections/e1d5c0c6-56c0-441e-b3e5-c48e1c51f1f1/query HTTP/1.1" 400
[32mINFO[0m:     [29-12-2023 19:01:15] Shutting down
[32mINFO[0m:     [29-12-2023 19:01:15] Waiting for application shutdown.
[32mINFO[0m:     [29-12-2023 19:01:15] Application shutdown complete.
[32mINFO[0m:     [29-12-2023 19:01:15] Finished server process [[36m3208[0m]
[32mINFO[0m:     [29-12-2023 19:01:21] Set chroma_server_nofile to 65535
[32mINFO[0m:     [29-12-2023 19:01:21] Anonymized telemetry enabled. See                     https://docs.trychroma.com/telemetry for more information.
[36mDEBUG[0m:    [29-12-2023 19:01:21] Starting component System
[36mDEBUG[0m:    [29-12-2023 19:01:21] Starting component OpenTelemetryClient
[36mDEBUG[0m:    [29-12-2023 19:01:21] Starting component SimpleAssignmentPolicy
[36mDEBUG[0m:    [29-12-2023 19:01:21] Starting component SqliteDB
[36mDEBUG[0m:    [29-12-2023 19:01:21] Starting component Posthog
[36mDEBUG[0m:    [29-12-2023 19:01:21] Starting component LocalSegmentManager
[36mDEBUG[0m:    [29-12-2023 19:01:21] Starting component SegmentAPI
[32mINFO[0m:     [29-12-2023 19:01:21] Started server process [[36m4303[0m]
[32mINFO[0m:     [29-12-2023 19:01:21] Waiting for application startup.
[32mINFO[0m:     [29-12-2023 19:01:21] Application startup complete.
[32mINFO[0m:     [29-12-2023 19:01:21] Uvicorn running on [1mhttp://localhost:8000[0m (Press CTRL+C to quit)
[32mINFO[0m:     [29-12-2023 19:01:28] ::1:54790 - "POST /api/v1/collections?tenant=default_tenant&database=default_database HTTP/1.1" 200
[32mINFO[0m:     [29-12-2023 19:01:28] ::1:54791 - "GET /api/v1/collections?tenant=default_tenant&database=default_database HTTP/1.1" 200
[32mINFO[0m:     [29-12-2023 19:01:28] ::1:54790 - "GET /api/v1/collections/movie_reviews?tenant=default_tenant&database=default_database HTTP/1.1" 200
[36mDEBUG[0m:    [29-12-2023 19:01:46] Starting component PersistentLocalHnswSegment
[32mINFO[0m:     [29-12-2023 19:01:46] ::1:54794 - "POST /api/v1/collections/d2e971b8-d30d-45b6-82c5-a17ee9626aa0/add HTTP/1.1" 201
[33mWARNING[0m:  [29-12-2023 19:01:46] Delete of nonexisting embedding ID: 5768bf05-f70e-4f6a-895d-a682ff9a99a9
[33mWARNING[0m:  [29-12-2023 19:01:46] Delete of nonexisting embedding ID: 5768bf05-f70e-4f6a-895d-a682ff9a99a9
[32mINFO[0m:     [29-12-2023 19:01:46] ::1:54795 - "POST /api/v1/collections/d2e971b8-d30d-45b6-82c5-a17ee9626aa0/delete HTTP/1.1" 200
[32mINFO[0m:     [29-12-2023 19:01:46] ::1:54794 - "POST /api/v1/collections/d2e971b8-d30d-45b6-82c5-a17ee9626aa0/query HTTP/1.1" 400
[32mINFO[0m:     [29-12-2023 19:03:27] Shutting down
[32mINFO[0m:     [29-12-2023 19:03:27] Waiting for application shutdown.
[32mINFO[0m:     [29-12-2023 19:03:27] Application shutdown complete.
[32mINFO[0m:     [29-12-2023 19:03:27] Finished server process [[36m4303[0m]
[32mINFO[0m:     [29-12-2023 19:03:34] Set chroma_server_nofile to 65535
[32mINFO[0m:     [29-12-2023 19:03:35] Anonymized telemetry enabled. See                     https://docs.trychroma.com/telemetry for more information.
[36mDEBUG[0m:    [29-12-2023 19:03:35] Starting component System
[36mDEBUG[0m:    [29-12-2023 19:03:35] Starting component OpenTelemetryClient
[36mDEBUG[0m:    [29-12-2023 19:03:35] Starting component SimpleAssignmentPolicy
[36mDEBUG[0m:    [29-12-2023 19:03:35] Starting component SqliteDB
[36mDEBUG[0m:    [29-12-2023 19:03:35] Starting component Posthog
[36mDEBUG[0m:    [29-12-2023 19:03:35] Starting component LocalSegmentManager
[36mDEBUG[0m:    [29-12-2023 19:03:35] Starting component SegmentAPI
[32mINFO[0m:     [29-12-2023 19:03:35] Started server process [[36m4670[0m]
[32mINFO[0m:     [29-12-2023 19:03:35] Waiting for application startup.
[32mINFO[0m:     [29-12-2023 19:03:35] Application startup complete.
[32mINFO[0m:     [29-12-2023 19:03:35] Uvicorn running on [1mhttp://localhost:8000[0m (Press CTRL+C to quit)
[32mINFO[0m:     [29-12-2023 19:03:46] ::1:54799 - "POST /api/v1/collections?tenant=default_tenant&database=default_database HTTP/1.1" 200
[32mINFO[0m:     [29-12-2023 19:03:46] ::1:54800 - "GET /api/v1/collections?tenant=default_tenant&database=default_database HTTP/1.1" 200
[32mINFO[0m:     [29-12-2023 19:03:46] ::1:54799 - "GET /api/v1/collections/movie_reviews?tenant=default_tenant&database=default_database HTTP/1.1" 200
[36mDEBUG[0m:    [29-12-2023 19:04:05] Starting component PersistentLocalHnswSegment
[32mINFO[0m:     [29-12-2023 19:04:05] ::1:54803 - "POST /api/v1/collections/b71978fe-c35f-4520-b63a-c6a1e3c2d0d2/add HTTP/1.1" 201
[33mWARNING[0m:  [29-12-2023 19:04:05] Delete of nonexisting embedding ID: 5768bf05-f70e-4f6a-895d-a682ff9a99a9
[33mWARNING[0m:  [29-12-2023 19:04:05] Delete of nonexisting embedding ID: 5768bf05-f70e-4f6a-895d-a682ff9a99a9
[32mINFO[0m:     [29-12-2023 19:04:05] ::1:54804 - "POST /api/v1/collections/b71978fe-c35f-4520-b63a-c6a1e3c2d0d2/delete HTTP/1.1" 200
[32mINFO[0m:     [29-12-2023 19:04:05] ::1:54803 - "POST /api/v1/collections/b71978fe-c35f-4520-b63a-c6a1e3c2d0d2/query HTTP/1.1" 400
[32mINFO[0m:     [29-12-2023 19:08:27] Shutting down
[32mINFO[0m:     [29-12-2023 19:08:27] Waiting for application shutdown.
[32mINFO[0m:     [29-12-2023 19:08:27] Application shutdown complete.
[32mINFO[0m:     [29-12-2023 19:08:27] Finished server process [[36m4670[0m]
[32mINFO[0m:     [29-12-2023 19:08:29] Set chroma_server_nofile to 65535
[32mINFO[0m:     [29-12-2023 19:08:30] Anonymized telemetry enabled. See                     https://docs.trychroma.com/telemetry for more information.
[36mDEBUG[0m:    [29-12-2023 19:08:30] Starting component System
[36mDEBUG[0m:    [29-12-2023 19:08:30] Starting component OpenTelemetryClient
[36mDEBUG[0m:    [29-12-2023 19:08:30] Starting component SimpleAssignmentPolicy
[36mDEBUG[0m:    [29-12-2023 19:08:30] Starting component SqliteDB
[36mDEBUG[0m:    [29-12-2023 19:08:30] Starting component Posthog
[36mDEBUG[0m:    [29-12-2023 19:08:30] Starting component LocalSegmentManager
[36mDEBUG[0m:    [29-12-2023 19:08:30] Starting component SegmentAPI
[32mINFO[0m:     [29-12-2023 19:08:30] Started server process [[36m5622[0m]
[32mINFO[0m:     [29-12-2023 19:08:30] Waiting for application startup.
[32mINFO[0m:     [29-12-2023 19:08:30] Application startup complete.
[32mINFO[0m:     [29-12-2023 19:08:30] Uvicorn running on [1mhttp://localhost:8000[0m (Press CTRL+C to quit)
[31mERROR[0m:    [29-12-2023 19:08:37] Collection movie_reviews already exists
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/anyio/streams/memory.py", line 94, in receive
    return self.receive_nowait()
  File "/usr/local/lib/python3.9/site-packages/anyio/streams/memory.py", line 89, in receive_nowait
    raise WouldBlock
anyio.WouldBlock

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/base.py", line 78, in call_next
    message = await recv_stream.receive()
  File "/usr/local/lib/python3.9/site-packages/anyio/streams/memory.py", line 114, in receive
    raise EndOfStream
anyio.EndOfStream

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/chromadb/server/fastapi/__init__.py", line 80, in catch_exceptions_middleware
    return await call_next(request)
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/base.py", line 84, in call_next
    raise app_exc
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/base.py", line 70, in coro
    await self.app(scope, receive_or_disconnect, send_no_error)
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/base.py", line 108, in __call__
    response = await self.dispatch_func(request, call_next)
  File "/usr/local/lib/python3.9/site-packages/chromadb/server/fastapi/__init__.py", line 94, in check_http_version_middleware
    return await call_next(request)
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/base.py", line 84, in call_next
    raise app_exc
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/base.py", line 70, in coro
    await self.app(scope, receive_or_disconnect, send_no_error)
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/exceptions.py", line 79, in __call__
    raise exc
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/exceptions.py", line 68, in __call__
    await self.app(scope, receive, sender)
  File "/usr/local/lib/python3.9/site-packages/fastapi/middleware/asyncexitstack.py", line 20, in __call__
    raise e
  File "/usr/local/lib/python3.9/site-packages/fastapi/middleware/asyncexitstack.py", line 17, in __call__
    await self.app(scope, receive, send)
  File "/usr/local/lib/python3.9/site-packages/starlette/routing.py", line 718, in __call__
    await route.handle(scope, receive, send)
  File "/usr/local/lib/python3.9/site-packages/starlette/routing.py", line 276, in handle
    await self.app(scope, receive, send)
  File "/usr/local/lib/python3.9/site-packages/starlette/routing.py", line 66, in app
    response = await func(request)
  File "/usr/local/lib/python3.9/site-packages/fastapi/routing.py", line 273, in app
    raw_response = await run_endpoint_function(
  File "/usr/local/lib/python3.9/site-packages/fastapi/routing.py", line 192, in run_endpoint_function
    return await run_in_threadpool(dependant.call, **values)
  File "/usr/local/lib/python3.9/site-packages/starlette/concurrency.py", line 41, in run_in_threadpool
    return await anyio.to_thread.run_sync(func, *args)
  File "/usr/local/lib/python3.9/site-packages/anyio/to_thread.py", line 31, in run_sync
    return await get_asynclib().run_sync_in_worker_thread(
  File "/usr/local/lib/python3.9/site-packages/anyio/_backends/_asyncio.py", line 937, in run_sync_in_worker_thread
    return await future
  File "/usr/local/lib/python3.9/site-packages/anyio/_backends/_asyncio.py", line 867, in run
    result = context.run(func, *args)
  File "/usr/local/lib/python3.9/site-packages/chromadb/telemetry/opentelemetry/__init__.py", line 127, in wrapper
    return f(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/chromadb/auth/fastapi.py", line 234, in wrapped
    return f(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/chromadb/server/fastapi/__init__.py", line 408, in create_collection
    return self._api.create_collection(
  File "/usr/local/lib/python3.9/site-packages/chromadb/telemetry/opentelemetry/__init__.py", line 127, in wrapper
    return f(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/chromadb/api/segment.py", line 166, in create_collection
    coll, created = self._sysdb.create_collection(
  File "/usr/local/lib/python3.9/site-packages/chromadb/telemetry/opentelemetry/__init__.py", line 127, in wrapper
    return f(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/chromadb/db/mixins/sysdb.py", line 225, in create_collection
    raise UniqueConstraintError(f"Collection {name} already exists")
chromadb.db.base.UniqueConstraintError: Collection movie_reviews already exists
[32mINFO[0m:     [29-12-2023 19:08:37] ::1:54831 - "POST /api/v1/collections?tenant=default_tenant&database=default_database HTTP/1.1" 500
[32mINFO[0m:     [29-12-2023 19:08:46] Shutting down
[32mINFO[0m:     [29-12-2023 19:08:46] Waiting for application shutdown.
[32mINFO[0m:     [29-12-2023 19:08:46] Application shutdown complete.
[32mINFO[0m:     [29-12-2023 19:08:46] Finished server process [[36m5622[0m]
[32mINFO[0m:     [29-12-2023 19:08:48] Set chroma_server_nofile to 65535
[32mINFO[0m:     [29-12-2023 19:08:48] Anonymized telemetry enabled. See                     https://docs.trychroma.com/telemetry for more information.
[36mDEBUG[0m:    [29-12-2023 19:08:48] Starting component System
[36mDEBUG[0m:    [29-12-2023 19:08:48] Starting component OpenTelemetryClient
[36mDEBUG[0m:    [29-12-2023 19:08:48] Starting component SimpleAssignmentPolicy
[36mDEBUG[0m:    [29-12-2023 19:08:48] Starting component SqliteDB
[36mDEBUG[0m:    [29-12-2023 19:08:48] Starting component Posthog
[36mDEBUG[0m:    [29-12-2023 19:08:48] Starting component LocalSegmentManager
[36mDEBUG[0m:    [29-12-2023 19:08:48] Starting component SegmentAPI
[32mINFO[0m:     [29-12-2023 19:08:48] Started server process [[36m5672[0m]
[32mINFO[0m:     [29-12-2023 19:08:48] Waiting for application startup.
[32mINFO[0m:     [29-12-2023 19:08:48] Application startup complete.
[32mINFO[0m:     [29-12-2023 19:08:48] Uvicorn running on [1mhttp://localhost:8000[0m (Press CTRL+C to quit)
[32mINFO[0m:     [29-12-2023 19:08:57] ::1:54833 - "POST /api/v1/collections?tenant=default_tenant&database=default_database HTTP/1.1" 200
[32mINFO[0m:     [29-12-2023 19:08:57] ::1:54834 - "GET /api/v1/collections?tenant=default_tenant&database=default_database HTTP/1.1" 200
[32mINFO[0m:     [29-12-2023 19:08:57] ::1:54833 - "GET /api/v1/collections/movie_reviews?tenant=default_tenant&database=default_database HTTP/1.1" 200
[36mDEBUG[0m:    [29-12-2023 19:09:17] Starting component PersistentLocalHnswSegment
[32mINFO[0m:     [29-12-2023 19:09:17] ::1:54839 - "POST /api/v1/collections/07e81729-ed71-4372-bb4d-ff281df89828/add HTTP/1.1" 201
[33mWARNING[0m:  [29-12-2023 19:09:17] Delete of nonexisting embedding ID: 5768bf05-f70e-4f6a-895d-a682ff9a99a9
[33mWARNING[0m:  [29-12-2023 19:09:17] Delete of nonexisting embedding ID: 5768bf05-f70e-4f6a-895d-a682ff9a99a9
[32mINFO[0m:     [29-12-2023 19:09:17] ::1:54840 - "POST /api/v1/collections/07e81729-ed71-4372-bb4d-ff281df89828/delete HTTP/1.1" 200
[32mINFO[0m:     [29-12-2023 19:09:17] ::1:54839 - "POST /api/v1/collections/07e81729-ed71-4372-bb4d-ff281df89828/query HTTP/1.1" 200
[32mINFO[0m:     [29-12-2023 19:20:45] Shutting down
[32mINFO[0m:     [29-12-2023 19:20:45] Waiting for application shutdown.
[32mINFO[0m:     [29-12-2023 19:20:45] Application shutdown complete.
[32mINFO[0m:     [29-12-2023 19:20:45] Finished server process [[36m5672[0m]
[32mINFO[0m:     [29-12-2023 19:20:52] Set chroma_server_nofile to 65535
[32mINFO[0m:     [29-12-2023 19:20:52] Anonymized telemetry enabled. See                     https://docs.trychroma.com/telemetry for more information.
[36mDEBUG[0m:    [29-12-2023 19:20:52] Starting component System
[36mDEBUG[0m:    [29-12-2023 19:20:52] Starting component OpenTelemetryClient
[36mDEBUG[0m:    [29-12-2023 19:20:52] Starting component SimpleAssignmentPolicy
[36mDEBUG[0m:    [29-12-2023 19:20:52] Starting component SqliteDB
[36mDEBUG[0m:    [29-12-2023 19:20:52] Starting component Posthog
[36mDEBUG[0m:    [29-12-2023 19:20:52] Starting component LocalSegmentManager
[36mDEBUG[0m:    [29-12-2023 19:20:52] Starting component SegmentAPI
[32mINFO[0m:     [29-12-2023 19:20:52] Started server process [[36m7670[0m]
[32mINFO[0m:     [29-12-2023 19:20:52] Waiting for application startup.
[32mINFO[0m:     [29-12-2023 19:20:52] Application startup complete.
[32mINFO[0m:     [29-12-2023 19:20:52] Uvicorn running on [1mhttp://localhost:8000[0m (Press CTRL+C to quit)
[32mINFO[0m:     [29-12-2023 19:20:56] ::1:54886 - "POST /api/v1/collections?tenant=default_tenant&database=default_database HTTP/1.1" 200
[32mINFO[0m:     [29-12-2023 19:20:56] ::1:54887 - "GET /api/v1/collections/movie_reviews?tenant=default_tenant&database=default_database HTTP/1.1" 200
[36mDEBUG[0m:    [29-12-2023 19:21:15] Starting component PersistentLocalHnswSegment
[32mINFO[0m:     [29-12-2023 19:21:15] ::1:54893 - "POST /api/v1/collections/8b29d140-29dc-4991-80f7-54742dff2320/add HTTP/1.1" 201
[33mWARNING[0m:  [29-12-2023 19:21:15] Delete of nonexisting embedding ID: 5768bf05-f70e-4f6a-895d-a682ff9a99a9
[33mWARNING[0m:  [29-12-2023 19:21:15] Delete of nonexisting embedding ID: 5768bf05-f70e-4f6a-895d-a682ff9a99a9
[32mINFO[0m:     [29-12-2023 19:21:15] ::1:54894 - "POST /api/v1/collections/8b29d140-29dc-4991-80f7-54742dff2320/delete HTTP/1.1" 200
[32mINFO[0m:     [29-12-2023 19:21:15] ::1:54893 - "POST /api/v1/collections/8b29d140-29dc-4991-80f7-54742dff2320/query HTTP/1.1" 200
[32mINFO[0m:     [29-12-2023 19:51:35] Shutting down
[32mINFO[0m:     [29-12-2023 19:51:35] Waiting for application shutdown.
[32mINFO[0m:     [29-12-2023 19:51:35] Application shutdown complete.
[32mINFO[0m:     [29-12-2023 19:51:35] Finished server process [[36m7670[0m]
[32mINFO[0m:     [29-12-2023 19:51:42] Set chroma_server_nofile to 65535
[32mINFO[0m:     [29-12-2023 19:51:42] Anonymized telemetry enabled. See                     https://docs.trychroma.com/telemetry for more information.
[36mDEBUG[0m:    [29-12-2023 19:51:42] Starting component System
[36mDEBUG[0m:    [29-12-2023 19:51:42] Starting component OpenTelemetryClient
[36mDEBUG[0m:    [29-12-2023 19:51:42] Starting component SimpleAssignmentPolicy
[36mDEBUG[0m:    [29-12-2023 19:51:42] Starting component SqliteDB
[36mDEBUG[0m:    [29-12-2023 19:51:42] Starting component Posthog
[36mDEBUG[0m:    [29-12-2023 19:51:42] Starting component LocalSegmentManager
[36mDEBUG[0m:    [29-12-2023 19:51:42] Starting component SegmentAPI
[32mINFO[0m:     [29-12-2023 19:51:42] Started server process [[36m12883[0m]
[32mINFO[0m:     [29-12-2023 19:51:42] Waiting for application startup.
[32mINFO[0m:     [29-12-2023 19:51:42] Application startup complete.
[32mINFO[0m:     [29-12-2023 19:51:42] Uvicorn running on [1mhttp://localhost:8000[0m (Press CTRL+C to quit)
[31mERROR[0m:    [29-12-2023 19:51:45] Collection movie_reviews already exists
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/anyio/streams/memory.py", line 94, in receive
    return self.receive_nowait()
  File "/usr/local/lib/python3.9/site-packages/anyio/streams/memory.py", line 89, in receive_nowait
    raise WouldBlock
anyio.WouldBlock

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/base.py", line 78, in call_next
    message = await recv_stream.receive()
  File "/usr/local/lib/python3.9/site-packages/anyio/streams/memory.py", line 114, in receive
    raise EndOfStream
anyio.EndOfStream

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/chromadb/server/fastapi/__init__.py", line 80, in catch_exceptions_middleware
    return await call_next(request)
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/base.py", line 84, in call_next
    raise app_exc
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/base.py", line 70, in coro
    await self.app(scope, receive_or_disconnect, send_no_error)
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/base.py", line 108, in __call__
    response = await self.dispatch_func(request, call_next)
  File "/usr/local/lib/python3.9/site-packages/chromadb/server/fastapi/__init__.py", line 94, in check_http_version_middleware
    return await call_next(request)
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/base.py", line 84, in call_next
    raise app_exc
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/base.py", line 70, in coro
    await self.app(scope, receive_or_disconnect, send_no_error)
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/exceptions.py", line 79, in __call__
    raise exc
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/exceptions.py", line 68, in __call__
    await self.app(scope, receive, sender)
  File "/usr/local/lib/python3.9/site-packages/fastapi/middleware/asyncexitstack.py", line 20, in __call__
    raise e
  File "/usr/local/lib/python3.9/site-packages/fastapi/middleware/asyncexitstack.py", line 17, in __call__
    await self.app(scope, receive, send)
  File "/usr/local/lib/python3.9/site-packages/starlette/routing.py", line 718, in __call__
    await route.handle(scope, receive, send)
  File "/usr/local/lib/python3.9/site-packages/starlette/routing.py", line 276, in handle
    await self.app(scope, receive, send)
  File "/usr/local/lib/python3.9/site-packages/starlette/routing.py", line 66, in app
    response = await func(request)
  File "/usr/local/lib/python3.9/site-packages/fastapi/routing.py", line 273, in app
    raw_response = await run_endpoint_function(
  File "/usr/local/lib/python3.9/site-packages/fastapi/routing.py", line 192, in run_endpoint_function
    return await run_in_threadpool(dependant.call, **values)
  File "/usr/local/lib/python3.9/site-packages/starlette/concurrency.py", line 41, in run_in_threadpool
    return await anyio.to_thread.run_sync(func, *args)
  File "/usr/local/lib/python3.9/site-packages/anyio/to_thread.py", line 31, in run_sync
    return await get_asynclib().run_sync_in_worker_thread(
  File "/usr/local/lib/python3.9/site-packages/anyio/_backends/_asyncio.py", line 937, in run_sync_in_worker_thread
    return await future
  File "/usr/local/lib/python3.9/site-packages/anyio/_backends/_asyncio.py", line 867, in run
    result = context.run(func, *args)
  File "/usr/local/lib/python3.9/site-packages/chromadb/telemetry/opentelemetry/__init__.py", line 127, in wrapper
    return f(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/chromadb/auth/fastapi.py", line 234, in wrapped
    return f(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/chromadb/server/fastapi/__init__.py", line 408, in create_collection
    return self._api.create_collection(
  File "/usr/local/lib/python3.9/site-packages/chromadb/telemetry/opentelemetry/__init__.py", line 127, in wrapper
    return f(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/chromadb/api/segment.py", line 166, in create_collection
    coll, created = self._sysdb.create_collection(
  File "/usr/local/lib/python3.9/site-packages/chromadb/telemetry/opentelemetry/__init__.py", line 127, in wrapper
    return f(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/chromadb/db/mixins/sysdb.py", line 225, in create_collection
    raise UniqueConstraintError(f"Collection {name} already exists")
chromadb.db.base.UniqueConstraintError: Collection movie_reviews already exists
[32mINFO[0m:     [29-12-2023 19:51:45] ::1:55027 - "POST /api/v1/collections?tenant=default_tenant&database=default_database HTTP/1.1" 500
[31mERROR[0m:    [29-12-2023 19:51:58] Collection movie_reviews already exists
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/anyio/streams/memory.py", line 94, in receive
    return self.receive_nowait()
  File "/usr/local/lib/python3.9/site-packages/anyio/streams/memory.py", line 89, in receive_nowait
    raise WouldBlock
anyio.WouldBlock

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/base.py", line 78, in call_next
    message = await recv_stream.receive()
  File "/usr/local/lib/python3.9/site-packages/anyio/streams/memory.py", line 114, in receive
    raise EndOfStream
anyio.EndOfStream

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/chromadb/server/fastapi/__init__.py", line 80, in catch_exceptions_middleware
    return await call_next(request)
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/base.py", line 84, in call_next
    raise app_exc
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/base.py", line 70, in coro
    await self.app(scope, receive_or_disconnect, send_no_error)
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/base.py", line 108, in __call__
    response = await self.dispatch_func(request, call_next)
  File "/usr/local/lib/python3.9/site-packages/chromadb/server/fastapi/__init__.py", line 94, in check_http_version_middleware
    return await call_next(request)
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/base.py", line 84, in call_next
    raise app_exc
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/base.py", line 70, in coro
    await self.app(scope, receive_or_disconnect, send_no_error)
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/exceptions.py", line 79, in __call__
    raise exc
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/exceptions.py", line 68, in __call__
    await self.app(scope, receive, sender)
  File "/usr/local/lib/python3.9/site-packages/fastapi/middleware/asyncexitstack.py", line 20, in __call__
    raise e
  File "/usr/local/lib/python3.9/site-packages/fastapi/middleware/asyncexitstack.py", line 17, in __call__
    await self.app(scope, receive, send)
  File "/usr/local/lib/python3.9/site-packages/starlette/routing.py", line 718, in __call__
    await route.handle(scope, receive, send)
  File "/usr/local/lib/python3.9/site-packages/starlette/routing.py", line 276, in handle
    await self.app(scope, receive, send)
  File "/usr/local/lib/python3.9/site-packages/starlette/routing.py", line 66, in app
    response = await func(request)
  File "/usr/local/lib/python3.9/site-packages/fastapi/routing.py", line 273, in app
    raw_response = await run_endpoint_function(
  File "/usr/local/lib/python3.9/site-packages/fastapi/routing.py", line 192, in run_endpoint_function
    return await run_in_threadpool(dependant.call, **values)
  File "/usr/local/lib/python3.9/site-packages/starlette/concurrency.py", line 41, in run_in_threadpool
    return await anyio.to_thread.run_sync(func, *args)
  File "/usr/local/lib/python3.9/site-packages/anyio/to_thread.py", line 31, in run_sync
    return await get_asynclib().run_sync_in_worker_thread(
  File "/usr/local/lib/python3.9/site-packages/anyio/_backends/_asyncio.py", line 937, in run_sync_in_worker_thread
    return await future
  File "/usr/local/lib/python3.9/site-packages/anyio/_backends/_asyncio.py", line 867, in run
    result = context.run(func, *args)
  File "/usr/local/lib/python3.9/site-packages/chromadb/telemetry/opentelemetry/__init__.py", line 127, in wrapper
    return f(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/chromadb/auth/fastapi.py", line 234, in wrapped
    return f(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/chromadb/server/fastapi/__init__.py", line 408, in create_collection
    return self._api.create_collection(
  File "/usr/local/lib/python3.9/site-packages/chromadb/telemetry/opentelemetry/__init__.py", line 127, in wrapper
    return f(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/chromadb/api/segment.py", line 166, in create_collection
    coll, created = self._sysdb.create_collection(
  File "/usr/local/lib/python3.9/site-packages/chromadb/telemetry/opentelemetry/__init__.py", line 127, in wrapper
    return f(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/chromadb/db/mixins/sysdb.py", line 225, in create_collection
    raise UniqueConstraintError(f"Collection {name} already exists")
chromadb.db.base.UniqueConstraintError: Collection movie_reviews already exists
[32mINFO[0m:     [29-12-2023 19:51:58] ::1:55030 - "POST /api/v1/collections?tenant=default_tenant&database=default_database HTTP/1.1" 500
[32mINFO[0m:     [29-12-2023 19:52:03] Shutting down
[32mINFO[0m:     [29-12-2023 19:52:03] Waiting for application shutdown.
[32mINFO[0m:     [29-12-2023 19:52:03] Application shutdown complete.
[32mINFO[0m:     [29-12-2023 19:52:03] Finished server process [[36m12883[0m]
[32mINFO[0m:     [29-12-2023 19:52:05] Set chroma_server_nofile to 65535
[32mINFO[0m:     [29-12-2023 19:52:05] Anonymized telemetry enabled. See                     https://docs.trychroma.com/telemetry for more information.
[36mDEBUG[0m:    [29-12-2023 19:52:05] Starting component System
[36mDEBUG[0m:    [29-12-2023 19:52:05] Starting component OpenTelemetryClient
[36mDEBUG[0m:    [29-12-2023 19:52:05] Starting component SimpleAssignmentPolicy
[36mDEBUG[0m:    [29-12-2023 19:52:05] Starting component SqliteDB
[36mDEBUG[0m:    [29-12-2023 19:52:05] Starting component Posthog
[36mDEBUG[0m:    [29-12-2023 19:52:05] Starting component LocalSegmentManager
[36mDEBUG[0m:    [29-12-2023 19:52:05] Starting component SegmentAPI
[32mINFO[0m:     [29-12-2023 19:52:06] Started server process [[36m12947[0m]
[32mINFO[0m:     [29-12-2023 19:52:06] Waiting for application startup.
[32mINFO[0m:     [29-12-2023 19:52:06] Application startup complete.
[32mINFO[0m:     [29-12-2023 19:52:06] Uvicorn running on [1mhttp://localhost:8000[0m (Press CTRL+C to quit)
[32mINFO[0m:     [29-12-2023 19:52:08] ::1:55032 - "POST /api/v1/collections?tenant=default_tenant&database=default_database HTTP/1.1" 200
[32mINFO[0m:     [29-12-2023 19:52:08] ::1:55033 - "GET /api/v1/collections/movie_reviews?tenant=default_tenant&database=default_database HTTP/1.1" 200
[32mINFO[0m:     [29-12-2023 19:53:53] Shutting down
[32mINFO[0m:     [29-12-2023 19:53:53] Waiting for application shutdown.
[32mINFO[0m:     [29-12-2023 19:53:53] Application shutdown complete.
[32mINFO[0m:     [29-12-2023 19:53:53] Finished server process [[36m12947[0m]
[32mINFO[0m:     [29-12-2023 19:53:59] Set chroma_server_nofile to 65535
[32mINFO[0m:     [29-12-2023 19:53:59] Anonymized telemetry enabled. See                     https://docs.trychroma.com/telemetry for more information.
[36mDEBUG[0m:    [29-12-2023 19:53:59] Starting component System
[36mDEBUG[0m:    [29-12-2023 19:53:59] Starting component OpenTelemetryClient
[36mDEBUG[0m:    [29-12-2023 19:53:59] Starting component SimpleAssignmentPolicy
[36mDEBUG[0m:    [29-12-2023 19:53:59] Starting component SqliteDB
[36mDEBUG[0m:    [29-12-2023 19:53:59] Starting component Posthog
[36mDEBUG[0m:    [29-12-2023 19:53:59] Starting component LocalSegmentManager
[36mDEBUG[0m:    [29-12-2023 19:53:59] Starting component SegmentAPI
[32mINFO[0m:     [29-12-2023 19:53:59] Started server process [[36m13264[0m]
[32mINFO[0m:     [29-12-2023 19:53:59] Waiting for application startup.
[32mINFO[0m:     [29-12-2023 19:53:59] Application startup complete.
[32mINFO[0m:     [29-12-2023 19:53:59] Uvicorn running on [1mhttp://localhost:8000[0m (Press CTRL+C to quit)
[32mINFO[0m:     [29-12-2023 19:54:02] ::1:55039 - "POST /api/v1/collections?tenant=default_tenant&database=default_database HTTP/1.1" 200
[32mINFO[0m:     [29-12-2023 19:54:02] ::1:55040 - "GET /api/v1/collections/movie_reviews?tenant=default_tenant&database=default_database HTTP/1.1" 200
[31mERROR[0m:    [29-12-2023 19:54:41] Collection movie_reviews already exists
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/anyio/streams/memory.py", line 94, in receive
    return self.receive_nowait()
  File "/usr/local/lib/python3.9/site-packages/anyio/streams/memory.py", line 89, in receive_nowait
    raise WouldBlock
anyio.WouldBlock

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/base.py", line 78, in call_next
    message = await recv_stream.receive()
  File "/usr/local/lib/python3.9/site-packages/anyio/streams/memory.py", line 114, in receive
    raise EndOfStream
anyio.EndOfStream

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/chromadb/server/fastapi/__init__.py", line 80, in catch_exceptions_middleware
    return await call_next(request)
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/base.py", line 84, in call_next
    raise app_exc
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/base.py", line 70, in coro
    await self.app(scope, receive_or_disconnect, send_no_error)
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/base.py", line 108, in __call__
    response = await self.dispatch_func(request, call_next)
  File "/usr/local/lib/python3.9/site-packages/chromadb/server/fastapi/__init__.py", line 94, in check_http_version_middleware
    return await call_next(request)
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/base.py", line 84, in call_next
    raise app_exc
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/base.py", line 70, in coro
    await self.app(scope, receive_or_disconnect, send_no_error)
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/exceptions.py", line 79, in __call__
    raise exc
  File "/usr/local/lib/python3.9/site-packages/starlette/middleware/exceptions.py", line 68, in __call__
    await self.app(scope, receive, sender)
  File "/usr/local/lib/python3.9/site-packages/fastapi/middleware/asyncexitstack.py", line 20, in __call__
    raise e
  File "/usr/local/lib/python3.9/site-packages/fastapi/middleware/asyncexitstack.py", line 17, in __call__
    await self.app(scope, receive, send)
  File "/usr/local/lib/python3.9/site-packages/starlette/routing.py", line 718, in __call__
    await route.handle(scope, receive, send)
  File "/usr/local/lib/python3.9/site-packages/starlette/routing.py", line 276, in handle
    await self.app(scope, receive, send)
  File "/usr/local/lib/python3.9/site-packages/starlette/routing.py", line 66, in app
    response = await func(request)
  File "/usr/local/lib/python3.9/site-packages/fastapi/routing.py", line 273, in app
    raw_response = await run_endpoint_function(
  File "/usr/local/lib/python3.9/site-packages/fastapi/routing.py", line 192, in run_endpoint_function
    return await run_in_threadpool(dependant.call, **values)
  File "/usr/local/lib/python3.9/site-packages/starlette/concurrency.py", line 41, in run_in_threadpool
    return await anyio.to_thread.run_sync(func, *args)
  File "/usr/local/lib/python3.9/site-packages/anyio/to_thread.py", line 31, in run_sync
    return await get_asynclib().run_sync_in_worker_thread(
  File "/usr/local/lib/python3.9/site-packages/anyio/_backends/_asyncio.py", line 937, in run_sync_in_worker_thread
    return await future
  File "/usr/local/lib/python3.9/site-packages/anyio/_backends/_asyncio.py", line 867, in run
    result = context.run(func, *args)
  File "/usr/local/lib/python3.9/site-packages/chromadb/telemetry/opentelemetry/__init__.py", line 127, in wrapper
    return f(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/chromadb/auth/fastapi.py", line 234, in wrapped
    return f(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/chromadb/server/fastapi/__init__.py", line 408, in create_collection
    return self._api.create_collection(
  File "/usr/local/lib/python3.9/site-packages/chromadb/telemetry/opentelemetry/__init__.py", line 127, in wrapper
    return f(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/chromadb/api/segment.py", line 166, in create_collection
    coll, created = self._sysdb.create_collection(
  File "/usr/local/lib/python3.9/site-packages/chromadb/telemetry/opentelemetry/__init__.py", line 127, in wrapper
    return f(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/chromadb/db/mixins/sysdb.py", line 225, in create_collection
    raise UniqueConstraintError(f"Collection {name} already exists")
chromadb.db.base.UniqueConstraintError: Collection movie_reviews already exists
[32mINFO[0m:     [29-12-2023 19:54:41] ::1:55042 - "POST /api/v1/collections?tenant=default_tenant&database=default_database HTTP/1.1" 500
[32mINFO[0m:     [29-12-2023 19:54:47] Shutting down
[32mINFO[0m:     [29-12-2023 19:54:47] Waiting for application shutdown.
[32mINFO[0m:     [29-12-2023 19:54:47] Application shutdown complete.
[32mINFO[0m:     [29-12-2023 19:54:47] Finished server process [[36m13264[0m]
[32mINFO[0m:     [29-12-2023 19:54:49] Set chroma_server_nofile to 65535
[32mINFO[0m:     [29-12-2023 19:54:49] Anonymized telemetry enabled. See                     https://docs.trychroma.com/telemetry for more information.
[36mDEBUG[0m:    [29-12-2023 19:54:49] Starting component System
[36mDEBUG[0m:    [29-12-2023 19:54:49] Starting component OpenTelemetryClient
[36mDEBUG[0m:    [29-12-2023 19:54:49] Starting component SimpleAssignmentPolicy
[36mDEBUG[0m:    [29-12-2023 19:54:49] Starting component SqliteDB
[36mDEBUG[0m:    [29-12-2023 19:54:49] Starting component Posthog
[36mDEBUG[0m:    [29-12-2023 19:54:49] Starting component LocalSegmentManager
[36mDEBUG[0m:    [29-12-2023 19:54:49] Starting component SegmentAPI
[32mINFO[0m:     [29-12-2023 19:54:49] Started server process [[36m13412[0m]
[32mINFO[0m:     [29-12-2023 19:54:49] Waiting for application startup.
[32mINFO[0m:     [29-12-2023 19:54:49] Application startup complete.
[32mINFO[0m:     [29-12-2023 19:54:49] Uvicorn running on [1mhttp://localhost:8000[0m (Press CTRL+C to quit)
[32mINFO[0m:     [29-12-2023 19:54:52] ::1:55045 - "POST /api/v1/collections?tenant=default_tenant&database=default_database HTTP/1.1" 200
[32mINFO[0m:     [29-12-2023 19:54:52] ::1:55046 - "GET /api/v1/collections/movie_reviews?tenant=default_tenant&database=default_database HTTP/1.1" 200
[36mDEBUG[0m:    [29-12-2023 19:55:12] Starting component PersistentLocalHnswSegment
[32mINFO[0m:     [29-12-2023 19:55:12] ::1:55051 - "POST /api/v1/collections/cec11f21-f0b1-470a-a7a3-1e52016266d7/add HTTP/1.1" 201
[33mWARNING[0m:  [29-12-2023 19:55:12] Delete of nonexisting embedding ID: 5768bf05-f70e-4f6a-895d-a682ff9a99a9
[33mWARNING[0m:  [29-12-2023 19:55:12] Delete of nonexisting embedding ID: 5768bf05-f70e-4f6a-895d-a682ff9a99a9
[32mINFO[0m:     [29-12-2023 19:55:12] ::1:55052 - "POST /api/v1/collections/cec11f21-f0b1-470a-a7a3-1e52016266d7/delete HTTP/1.1" 200
[32mINFO[0m:     [29-12-2023 19:55:12] ::1:55051 - "POST /api/v1/collections/cec11f21-f0b1-470a-a7a3-1e52016266d7/query HTTP/1.1" 200
[32mINFO[0m:     [29-12-2023 19:58:36] Shutting down
[32mINFO[0m:     [29-12-2023 19:58:36] Waiting for application shutdown.
[32mINFO[0m:     [29-12-2023 19:58:36] Application shutdown complete.
[32mINFO[0m:     [29-12-2023 19:58:36] Finished server process [[36m13412[0m]
[32mINFO[0m:     [29-12-2023 19:59:14] Set chroma_server_nofile to 65535
[32mINFO[0m:     [29-12-2023 19:59:14] Anonymized telemetry enabled. See                     https://docs.trychroma.com/telemetry for more information.
[36mDEBUG[0m:    [29-12-2023 19:59:14] Starting component System
[36mDEBUG[0m:    [29-12-2023 19:59:14] Starting component OpenTelemetryClient
[36mDEBUG[0m:    [29-12-2023 19:59:14] Starting component SimpleAssignmentPolicy
[36mDEBUG[0m:    [29-12-2023 19:59:14] Starting component SqliteDB
[36mDEBUG[0m:    [29-12-2023 19:59:14] Starting component Posthog
[36mDEBUG[0m:    [29-12-2023 19:59:14] Starting component LocalSegmentManager
[36mDEBUG[0m:    [29-12-2023 19:59:14] Starting component SegmentAPI
[32mINFO[0m:     [29-12-2023 19:59:14] Started server process [[36m14237[0m]
[32mINFO[0m:     [29-12-2023 19:59:14] Waiting for application startup.
[32mINFO[0m:     [29-12-2023 19:59:14] Application startup complete.
[32mINFO[0m:     [29-12-2023 19:59:14] Uvicorn running on [1mhttp://localhost:8000[0m (Press CTRL+C to quit)
[32mINFO[0m:     [29-12-2023 19:59:19] ::1:55067 - "POST /api/v1/collections?tenant=default_tenant&database=default_database HTTP/1.1" 200
[32mINFO[0m:     [29-12-2023 19:59:19] ::1:55068 - "GET /api/v1/collections/movie_reviews?tenant=default_tenant&database=default_database HTTP/1.1" 200
[36mDEBUG[0m:    [29-12-2023 19:59:38] Starting component PersistentLocalHnswSegment
[32mINFO[0m:     [29-12-2023 19:59:38] ::1:55072 - "POST /api/v1/collections/7d67acfc-558d-4cfc-bd02-508f927ef39b/add HTTP/1.1" 201
[33mWARNING[0m:  [29-12-2023 19:59:38] Delete of nonexisting embedding ID: 5768bf05-f70e-4f6a-895d-a682ff9a99a9
[33mWARNING[0m:  [29-12-2023 19:59:38] Delete of nonexisting embedding ID: 5768bf05-f70e-4f6a-895d-a682ff9a99a9
[32mINFO[0m:     [29-12-2023 19:59:38] ::1:55073 - "POST /api/v1/collections/7d67acfc-558d-4cfc-bd02-508f927ef39b/delete HTTP/1.1" 200
[32mINFO[0m:     [29-12-2023 19:59:38] ::1:55072 - "POST /api/v1/collections/7d67acfc-558d-4cfc-bd02-508f927ef39b/query HTTP/1.1" 200
[32mINFO[0m:     [29-12-2023 20:01:44] Shutting down
[32mINFO[0m:     [29-12-2023 20:01:44] Waiting for application shutdown.
[32mINFO[0m:     [29-12-2023 20:01:44] Application shutdown complete.
[32mINFO[0m:     [29-12-2023 20:01:44] Finished server process [[36m14237[0m]
[32mINFO[0m:     [29-12-2023 20:01:49] Set chroma_server_nofile to 65535
[32mINFO[0m:     [29-12-2023 20:01:49] Anonymized telemetry enabled. See                     https://docs.trychroma.com/telemetry for more information.
[36mDEBUG[0m:    [29-12-2023 20:01:49] Starting component System
[36mDEBUG[0m:    [29-12-2023 20:01:49] Starting component OpenTelemetryClient
[36mDEBUG[0m:    [29-12-2023 20:01:49] Starting component SimpleAssignmentPolicy
[36mDEBUG[0m:    [29-12-2023 20:01:49] Starting component SqliteDB
[36mDEBUG[0m:    [29-12-2023 20:01:49] Starting component Posthog
[36mDEBUG[0m:    [29-12-2023 20:01:49] Starting component LocalSegmentManager
[36mDEBUG[0m:    [29-12-2023 20:01:49] Starting component SegmentAPI
[32mINFO[0m:     [29-12-2023 20:01:49] Started server process [[36m14675[0m]
[32mINFO[0m:     [29-12-2023 20:01:49] Waiting for application startup.
[32mINFO[0m:     [29-12-2023 20:01:49] Application startup complete.
[32mINFO[0m:     [29-12-2023 20:01:49] Uvicorn running on [1mhttp://localhost:8000[0m (Press CTRL+C to quit)
[32mINFO[0m:     [29-12-2023 20:01:52] ::1:55082 - "POST /api/v1/collections?tenant=default_tenant&database=default_database HTTP/1.1" 200
[32mINFO[0m:     [29-12-2023 20:01:52] ::1:55083 - "GET /api/v1/collections/movie_reviews?tenant=default_tenant&database=default_database HTTP/1.1" 200
[36mDEBUG[0m:    [29-12-2023 20:02:14] Starting component PersistentLocalHnswSegment
[32mINFO[0m:     [29-12-2023 20:02:14] ::1:55086 - "POST /api/v1/collections/8b075da5-02f6-4a10-97bf-6241683385ad/add HTTP/1.1" 201
[33mWARNING[0m:  [29-12-2023 20:02:14] Delete of nonexisting embedding ID: 5768bf05-f70e-4f6a-895d-a682ff9a99a9
[33mWARNING[0m:  [29-12-2023 20:02:14] Delete of nonexisting embedding ID: 5768bf05-f70e-4f6a-895d-a682ff9a99a9
[32mINFO[0m:     [29-12-2023 20:02:14] ::1:55087 - "POST /api/v1/collections/8b075da5-02f6-4a10-97bf-6241683385ad/delete HTTP/1.1" 200
[32mINFO[0m:     [29-12-2023 20:02:14] ::1:55086 - "POST /api/v1/collections/8b075da5-02f6-4a10-97bf-6241683385ad/query HTTP/1.1" 200
